# Chapter Summary

- Individuals often mate non-randomly, e.g. by geographical location, this generates population genetic structure that can be thought of as a form of inbreeding. This inbreeding at a population level leads to a reduction in heterozygosity within sub-populations as compared to the total population (if allele frequencies differ across populations).

- Wright's $F$ statistics can be used to measure the extent of population structure, describing the reduction in heterozygosity at various scales, for example the individual compared to the sub-population ($F_{IS}$) or the sub-population compared to the total population ($F_{ST}$). We can calculate these statistics either genome-wide or at individual loci.

- These $F$ statistics can be understood as expressing a correlation between alleles drawn from the same level of population structure, or the proportion of genetic variance explained by population structure.

- Other ways to visualize population structure include STRUCTURE-like approaches, which are based on assigning individuals to populations based on the likelihood of their genotype given allele frequencies (assignment methods) and learning the assignment of individuals to discrete populations. Another common approach relies on identifying major axes of variation in relatedness via Principal components analysis.

- We'll often be interested in covariances and correlations among alleles at different loci, linkage disequilibrium (LD).

- Covariance between loci (LD) can arise between loci for a variety of reasons, notably population structure and admixture as described in the chapter.

- The decay of LD due recombination can be modelled and potentially used to date when LD was generated (e.g. via admixture).