
<!DOCTYPE html>

<html>
  <head>
    <meta charset="utf-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" /><meta name="generator" content="Docutils 0.17.1: http://docutils.sourceforge.net/" />

    <title>The coalescent process of a sample of alleles &#8212; Population And Quantitative Genetics</title>
    
  <!-- Loaded before other Sphinx assets -->
  <link href="../../_static/styles/theme.css?digest=1999514e3f237ded88cf" rel="stylesheet">
<link href="../../_static/styles/pydata-sphinx-theme.css?digest=1999514e3f237ded88cf" rel="stylesheet">

    
  <link rel="stylesheet"
    href="../../_static/vendor/fontawesome/5.13.0/css/all.min.css">
  <link rel="preload" as="font" type="font/woff2" crossorigin
    href="../../_static/vendor/fontawesome/5.13.0/webfonts/fa-solid-900.woff2">
  <link rel="preload" as="font" type="font/woff2" crossorigin
    href="../../_static/vendor/fontawesome/5.13.0/webfonts/fa-brands-400.woff2">

    <link rel="stylesheet" type="text/css" href="../../_static/pygments.css" />
    <link rel="stylesheet" href="../../_static/styles/sphinx-book-theme.css?digest=5115cc725059bd94278eecd172e13a965bf8f5a9" type="text/css" />
    <link rel="stylesheet" type="text/css" href="../../_static/togglebutton.css" />
    <link rel="stylesheet" type="text/css" href="../../_static/copybutton.css" />
    <link rel="stylesheet" type="text/css" href="../../_static/mystnb.css" />
    <link rel="stylesheet" type="text/css" href="../../_static/sphinx-thebe.css" />
    <link rel="stylesheet" type="text/css" href="../../_static/design-style.b7bb847fb20b106c3d81b95245e65545.min.css" />
    
  <!-- Pre-loaded scripts that we'll load fully later -->
  <link rel="preload" as="script" href="../../_static/scripts/pydata-sphinx-theme.js?digest=1999514e3f237ded88cf">

    <script data-url_root="../../" id="documentation_options" src="../../_static/documentation_options.js"></script>
    <script src="../../_static/jquery.js"></script>
    <script src="../../_static/underscore.js"></script>
    <script src="../../_static/doctools.js"></script>
    <script src="../../_static/clipboard.min.js"></script>
    <script src="../../_static/copybutton.js"></script>
    <script src="../../_static/scripts/sphinx-book-theme.js?digest=9c920249402e914e316237a7dbc6769907cce411"></script>
    <script>let toggleHintShow = 'Click to show';</script>
    <script>let toggleHintHide = 'Click to hide';</script>
    <script>let toggleOpenOnPrint = 'true';</script>
    <script src="../../_static/togglebutton.js"></script>
    <script>var togglebuttonSelector = '.toggle, .admonition.dropdown, .tag_hide_input div.cell_input, .tag_hide-input div.cell_input, .tag_hide_output div.cell_output, .tag_hide-output div.cell_output, .tag_hide_cell.cell, .tag_hide-cell.cell';</script>
    <script src="../../_static/design-tabs.js"></script>
    <script>const THEBE_JS_URL = "https://unpkg.com/thebe@0.8.2/lib/index.js"
const thebe_selector = ".thebe,.cell"
const thebe_selector_input = "pre"
const thebe_selector_output = ".output, .cell_output"
</script>
    <script async="async" src="../../_static/sphinx-thebe.js"></script>
    <script>window.MathJax = {"options": {"processHtmlClass": "tex2jax_process|mathjax_process|math|output_area"}}</script>
    <script defer="defer" src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-mml-chtml.js"></script>
    <link rel="index" title="Index" href="../../genindex.html" />
    <link rel="search" title="Search" href="../../search.html" />
    <link rel="next" title="Chapter Summary" href="chapter-04.5-chapter-summary.html" />
    <link rel="prev" title="The Coalescent and patterns of neutral diversity" href="chapter-04.3-the-coalescent-and-patterns-of-neutral-diversity.html" />
    <meta name="viewport" content="width=device-width, initial-scale=1" />
    <meta name="docsearch:language" content="None">
    

    <!-- Google Analytics -->
    
  </head>
  <body data-spy="scroll" data-target="#bd-toc-nav" data-offset="60">
<!-- Checkboxes to toggle the left sidebar -->
<input type="checkbox" class="sidebar-toggle" name="__navigation" id="__navigation" aria-label="Toggle navigation sidebar">
<label class="overlay overlay-navbar" for="__navigation">
    <div class="visually-hidden">Toggle navigation sidebar</div>
</label>
<!-- Checkboxes to toggle the in-page toc -->
<input type="checkbox" class="sidebar-toggle" name="__page-toc" id="__page-toc" aria-label="Toggle in-page Table of Contents">
<label class="overlay overlay-pagetoc" for="__page-toc">
    <div class="visually-hidden">Toggle in-page Table of Contents</div>
</label>
<!-- Headers at the top -->
<div class="announcement header-item noprint"></div>
<div class="header header-item noprint"></div>

    
    <div class="container-fluid" id="banner"></div>

    

    <div class="container-xl">
      <div class="row">
          
<!-- Sidebar -->
<div class="bd-sidebar noprint" id="site-navigation">
    <div class="bd-sidebar__content">
        <div class="bd-sidebar__top"><div class="navbar-brand-box">
    <a class="navbar-brand text-wrap" href="../../index.html">
      
        <!-- `logo` is deprecated in Sphinx 4.0, so remove this when we stop supporting 3 -->
        
      
      
      <img src="../../_static/logo.png" class="logo" alt="logo">
      
      
      <h1 class="site-logo" id="site-title">Population And Quantitative Genetics</h1>
      
    </a>
</div><form class="bd-search d-flex align-items-center" action="../../search.html" method="get">
  <i class="icon fas fa-search"></i>
  <input type="search" class="form-control" name="q" id="search-input" placeholder="Search this book..." aria-label="Search this book..." autocomplete="off" >
</form><nav class="bd-links" id="bd-docs-nav" aria-label="Main">
    <div class="bd-toc-item active">
        
        <ul class="nav bd-sidenav bd-sidenav__home-link">
            <li class="toctree-l1">
                <a class="reference internal" href="../chapter-01-introduction.html">
                    Introduction
                </a>
            </li>
        </ul>
        <ul class="current nav bd-sidenav">
 <li class="toctree-l1 has-children">
  <a class="reference internal" href="../chapter-02/chapter-02.1-allele-and-genotype-frequencies.html">
   Allele And Genotype Frequencies
  </a>
  <input class="toctree-checkbox" id="toctree-checkbox-1" name="toctree-checkbox-1" type="checkbox"/>
  <label for="toctree-checkbox-1">
   <i class="fas fa-chevron-down">
   </i>
  </label>
  <ul>
   <li class="toctree-l2">
    <a class="reference internal" href="../chapter-02/chapter-02.2-allele-frequencies.html">
     Allele Frequencies
    </a>
   </li>
   <li class="toctree-l2">
    <a class="reference internal" href="../chapter-02/chapter-02.3-allele-sharing-among-related-individuals-and-identity-by-descent.html">
     Allele Sharing Among Related Individuals And Identity By Descent
    </a>
   </li>
   <li class="toctree-l2">
    <a class="reference internal" href="../chapter-02/chapter-02.4-chapter-summary.html">
     Chapter Summary
    </a>
   </li>
   <li class="toctree-l2">
    <a class="reference internal" href="../chapter-02/chapter-02.5-further-questions.html">
     Further Questions
    </a>
   </li>
  </ul>
 </li>
 <li class="toctree-l1 has-children">
  <a class="reference internal" href="../chapter-03/chapter-03.1-population-structure-and-correlations-among-loci.html">
   Population Structure and Correlations Among Loci
  </a>
  <input class="toctree-checkbox" id="toctree-checkbox-2" name="toctree-checkbox-2" type="checkbox"/>
  <label for="toctree-checkbox-2">
   <i class="fas fa-chevron-down">
   </i>
  </label>
  <ul>
   <li class="toctree-l2">
    <a class="reference internal" href="../chapter-03/chapter-03.2-chapter-summary.html">
     Chapter Summary
    </a>
   </li>
   <li class="toctree-l2">
    <a class="reference internal" href="../chapter-03/chapter-03.3-further-questions.html">
     Further Questions
    </a>
   </li>
  </ul>
 </li>
 <li class="toctree-l1 current active has-children">
  <a class="reference internal" href="chapter-04.1-genetic-drift-and-neutral-diversity.html">
   Genetic Drift and Neutral Diversity
  </a>
  <input checked="" class="toctree-checkbox" id="toctree-checkbox-3" name="toctree-checkbox-3" type="checkbox"/>
  <label for="toctree-checkbox-3">
   <i class="fas fa-chevron-down">
   </i>
  </label>
  <ul class="current">
   <li class="toctree-l2">
    <a class="reference internal" href="chapter-04.2-loss-of-heterozygosity-due-to-drift.html">
     Loss of heterozygosity due to drift
    </a>
   </li>
   <li class="toctree-l2">
    <a class="reference internal" href="chapter-04.3-the-coalescent-and-patterns-of-neutral-diversity.html">
     The Coalescent and patterns of neutral diversity
    </a>
   </li>
   <li class="toctree-l2 current active">
    <a class="current reference internal" href="#">
     The coalescent process of a sample of alleles
    </a>
   </li>
   <li class="toctree-l2">
    <a class="reference internal" href="chapter-04.5-chapter-summary.html">
     Chapter Summary
    </a>
   </li>
   <li class="toctree-l2">
    <a class="reference internal" href="chapter-04.6-further-questions.html">
     Further Questions
    </a>
   </li>
  </ul>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../chapter-00-bibliography.html">
   Bibliography
  </a>
 </li>
</ul>

    </div>
</nav></div>
        <div class="bd-sidebar__bottom">
             <!-- To handle the deprecated key -->
            
            <div class="navbar_extra_footer">
            Powered by <a href="https://jupyterbook.org">Jupyter Book</a>
            </div>
            
        </div>
    </div>
    <div id="rtd-footer-container"></div>
</div>


          


          
<!-- A tiny helper pixel to detect if we've scrolled -->
<div class="sbt-scroll-pixel-helper"></div>
<!-- Main content -->
<div class="col py-0 content-container">
    
    <div class="header-article row sticky-top noprint">
        



<div class="col py-1 d-flex header-article-main">
    <div class="header-article__left">
        
        <label for="__navigation"
  class="headerbtn"
  data-toggle="tooltip"
data-placement="right"
title="Toggle navigation"
>
  

<span class="headerbtn__icon-container">
  <i class="fas fa-bars"></i>
  </span>

</label>

        
    </div>
    <div class="header-article__right">
<button onclick="toggleFullScreen()"
  class="headerbtn"
  data-toggle="tooltip"
data-placement="bottom"
title="Fullscreen mode"
>
  

<span class="headerbtn__icon-container">
  <i class="fas fa-expand"></i>
  </span>

</button>

<div class="menu-dropdown menu-dropdown-repository-buttons">
  <button class="headerbtn menu-dropdown__trigger"
      aria-label="Source repositories">
      <i class="fab fa-github"></i>
  </button>
  <div class="menu-dropdown__content">
    <ul>
      <li>
        <a href="https://github.com/executablebooks/jupyter-book"
   class="headerbtn"
   data-toggle="tooltip"
data-placement="left"
title="Source repository"
>
  

<span class="headerbtn__icon-container">
  <i class="fab fa-github"></i>
  </span>
<span class="headerbtn__text-container">repository</span>
</a>

      </li>
      
      <li>
        <a href="https://github.com/executablebooks/jupyter-book/issues/new?title=Issue%20on%20page%20%2Fchapters/chapter-04/chapter-04.4-the-coalescent-process-of-a-sample-of-alleles.html&body=Your%20issue%20content%20here."
   class="headerbtn"
   data-toggle="tooltip"
data-placement="left"
title="Open an issue"
>
  

<span class="headerbtn__icon-container">
  <i class="fas fa-lightbulb"></i>
  </span>
<span class="headerbtn__text-container">open issue</span>
</a>

      </li>
      
    </ul>
  </div>
</div>

<div class="menu-dropdown menu-dropdown-download-buttons">
  <button class="headerbtn menu-dropdown__trigger"
      aria-label="Download this page">
      <i class="fas fa-download"></i>
  </button>
  <div class="menu-dropdown__content">
    <ul>
      <li>
        <a href="../../_sources/chapters/chapter-04/chapter-04.4-the-coalescent-process-of-a-sample-of-alleles.md"
   class="headerbtn"
   data-toggle="tooltip"
data-placement="left"
title="Download source file"
>
  

<span class="headerbtn__icon-container">
  <i class="fas fa-file"></i>
  </span>
<span class="headerbtn__text-container">.md</span>
</a>

      </li>
      
      <li>
        
<button onclick="printPdf(this)"
  class="headerbtn"
  data-toggle="tooltip"
data-placement="left"
title="Print to PDF"
>
  

<span class="headerbtn__icon-container">
  <i class="fas fa-file-pdf"></i>
  </span>
<span class="headerbtn__text-container">.pdf</span>
</button>

      </li>
      
    </ul>
  </div>
</div>
<label for="__page-toc"
  class="headerbtn headerbtn-page-toc"
  
>
  

<span class="headerbtn__icon-container">
  <i class="fas fa-list"></i>
  </span>

</label>

    </div>
</div>

<!-- Table of contents -->
<div class="col-md-3 bd-toc show noprint">
    <div class="tocsection onthispage pt-5 pb-3">
        <i class="fas fa-list"></i> Contents
    </div>
    <nav id="bd-toc-nav" aria-label="Page">
        <ul class="visible nav section-nav flex-column">
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#simulating-a-coalescent-genealogy">
   Simulating a coalescent genealogy
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#expected-properties-of-coalescent-genealogies-and-mutations">
   Expected properties of coalescent genealogies and mutations
  </a>
  <ul class="nav section-nav flex-column">
   <li class="toc-h3 nav-item toc-entry">
    <a class="reference internal nav-link" href="#the-expected-time-to-the-most-recent-common-ancestor">
     The expected time to the most recent common ancestor
    </a>
   </li>
   <li class="toc-h3 nav-item toc-entry">
    <a class="reference internal nav-link" href="#the-expected-total-time-in-a-genealogy-and-the-number-of-segregating-sites">
     The expected total time in a genealogy and the number of segregating sites
    </a>
   </li>
   <li class="toc-h3 nav-item toc-entry">
    <a class="reference internal nav-link" href="#the-neutral-site-frequency-spectrum">
     The neutral site-frequency spectrum
    </a>
   </li>
   <li class="toc-h3 nav-item toc-entry">
    <a class="reference internal nav-link" href="#tests-based-on-the-site-frequency-spectrum">
     Tests based on the site frequency spectrum
    </a>
   </li>
  </ul>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#demography-and-the-coalescent">
   Demography and the coalescent
  </a>
 </li>
</ul>

    </nav>
</div>
    </div>
    <div class="article row">
        <div class="col pl-md-3 pl-lg-5 content-container">
            <!-- Table of contents that is only displayed when printing the page -->
            <div id="jb-print-docs-body" class="onlyprint">
                <h1>The coalescent process of a sample of alleles</h1>
                <!-- Table of contents -->
                <div id="print-main-content">
                    <div id="jb-print-toc">
                        
                        <div>
                            <h2> Contents </h2>
                        </div>
                        <nav aria-label="Page">
                            <ul class="visible nav section-nav flex-column">
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#simulating-a-coalescent-genealogy">
   Simulating a coalescent genealogy
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#expected-properties-of-coalescent-genealogies-and-mutations">
   Expected properties of coalescent genealogies and mutations
  </a>
  <ul class="nav section-nav flex-column">
   <li class="toc-h3 nav-item toc-entry">
    <a class="reference internal nav-link" href="#the-expected-time-to-the-most-recent-common-ancestor">
     The expected time to the most recent common ancestor
    </a>
   </li>
   <li class="toc-h3 nav-item toc-entry">
    <a class="reference internal nav-link" href="#the-expected-total-time-in-a-genealogy-and-the-number-of-segregating-sites">
     The expected total time in a genealogy and the number of segregating sites
    </a>
   </li>
   <li class="toc-h3 nav-item toc-entry">
    <a class="reference internal nav-link" href="#the-neutral-site-frequency-spectrum">
     The neutral site-frequency spectrum
    </a>
   </li>
   <li class="toc-h3 nav-item toc-entry">
    <a class="reference internal nav-link" href="#tests-based-on-the-site-frequency-spectrum">
     Tests based on the site frequency spectrum
    </a>
   </li>
  </ul>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#demography-and-the-coalescent">
   Demography and the coalescent
  </a>
 </li>
</ul>

                        </nav>
                    </div>
                </div>
            </div>
            <main id="main-content" role="main">
                
              <div>
                
  <section class="tex2jax_ignore mathjax_ignore" id="the-coalescent-process-of-a-sample-of-alleles">
<h1>The coalescent process of a sample of alleles<a class="headerlink" href="#the-coalescent-process-of-a-sample-of-alleles" title="Permalink to this headline">#</a></h1>
<p>Usually we are not just interested in pairs of alleles, or the average
pairwise diversity. Generally we are interested in the properties of
diversity in samples of a number of alleles drawn from the population.
Instead of just following a pair of lineages back until they coalesce,
we can follow the history of a sample of alleles back through the
population.</p>
<p>Consider first sampling three alleles at random from the population. The
probability that all three alleles choose exactly the same ancestral
allele one generation back is <span class="math notranslate nohighlight">\(\frac{1}{(2N)^2}\)</span>. If <span class="math notranslate nohighlight">\(N\)</span> is
reasonably large, then this is a very small probability. As such, it is
very unlikely that our three alleles coalesce all at once, and in a
moment we’ll see that it is safe to ignore such unlikely events.</p>
<figure class="align-left" id="figure-4-18">
<img alt="../../_images/Coal_three_lineages.pdf" src="../../_images/Coal_three_lineages.pdf" />
<figcaption>
<p><span class="caption-number">Fig. 43 </span><span class="caption-text">- A simple simulation of the coalescent process for three lineages. We
track the ancestry of three modern-day alleles, the first pair (red and
purple) coalesce four generations back, after which there are only two
independent lineages we are tracking. This pair then coalesces twelve
generations in the past. Note that different random realizations of this
process will differ from each other a lot. The <span class="math notranslate nohighlight">\(T_{MRCA}\)</span> is <span class="math notranslate nohighlight">\(T_3+T_2\)</span>.
The total time in the tree is <span class="math notranslate nohighlight">\(T_{tot}=3T_3 + 2T_2= 25\)</span> generations.</span><a class="headerlink" href="#figure-4-18" title="Permalink to this image">#</a></p>
</figcaption>
</figure>
<p>The probability that a specific pair of alleles find a common ancestor
in the preceding generation is still <span class="math notranslate nohighlight">\(\frac{1}{(2N)}\)</span>. There are
three possible pairs of alleles, so the probability that no pair finds a
common ancestor in the preceding generation is</p>
<div class="math notranslate nohighlight" id="equation-eq-4-28">
<span class="eqno">(58)<a class="headerlink" href="#equation-eq-4-28" title="Permalink to this equation">#</a></span>\[    \left(1-\frac{1}{2N} \right)^3 \approx \left( 1- \frac{3}{2N} \right)\]</div>
<p>In making this approximation we are multiplying out the right hand-side
and ignoring terms of <span class="math notranslate nohighlight">\(1/N^2\)</span> and higher (a Taylor approximation, see
Appendix equation <code class="xref eq docutils literal notranslate"><span class="pre">eq-A.2</span></code>). See <a class="reference internal" href="#figure-4-18"><span class="std std-numref">Fig. 43</span></a> for a random realization of
this process.</p>
<p>More generally, when we sample <span class="math notranslate nohighlight">\(i\)</span> alleles there are <span class="math notranslate nohighlight">\({i \choose 2}\)</span>
pairs, i.e. <span class="math notranslate nohighlight">\(i(i-1)/2\)</span> pairs. Thus, the probability that no pair of
alleles in a sample of size <span class="math notranslate nohighlight">\(i\)</span> coalesces in the preceding generation is</p>
<div class="math notranslate nohighlight" id="equation-eq-4-29">
<span class="eqno">(59)<a class="headerlink" href="#equation-eq-4-29" title="Permalink to this equation">#</a></span>\[    \left(1-\frac{1}{(2N)} \right)^{{i \choose
 2}} \approx \left( 1- \frac{{i \choose
 2}}{2N}\right)\]</div>
<p>while the probability any pair coalesces is
<span class="math notranslate nohighlight">\(\approx \frac{{i \choose
 2}}{2N}\)</span>, again using Appendix equation <code class="xref eq docutils literal notranslate"><span class="pre">eq-A.2</span></code>.</p>
<p>We can ignore the possibility that more than pairs of alleles (e.g.
tripletons) simultaneously coalesce at once as terms of
<span class="math notranslate nohighlight">\(\frac{1}{N^2}\)</span> and higher can be ignored as they are vanishingly
rare. Obviously in reasonable sample sizes there are many more triples
(<span class="math notranslate nohighlight">\({i \choose 3}\)</span>) and higher order combinations than there are pairs
(<span class="math notranslate nohighlight">\({i \choose 2}\)</span>), but if <span class="math notranslate nohighlight">\(i \ll N\)</span> then we are safe to ignore these
terms.</p>
<p>When there are <span class="math notranslate nohighlight">\(i\)</span> alleles, the probability that we wait until the <span class="math notranslate nohighlight">\(t+1\)</span>
generation before any pair of alleles coalesces is</p>
<div class="math notranslate nohighlight" id="equation-eq-4-30">
<span class="eqno">(60)<a class="headerlink" href="#equation-eq-4-30" title="Permalink to this equation">#</a></span>\[    P(T_i =t+1) = \frac{{i \choose
 2}}{2N}\left( 1- \frac{{i \choose
 2}}{2N}\right)^{t}\]</div>
<p>Thus the waiting time to the first
coalescent event while there are <span class="math notranslate nohighlight">\(i\)</span> lineages is a geometrically
distributed random variable with probability of success
<span class="math notranslate nohighlight">\(p=\frac{{i \choose 2}}{2N}\)</span>, which we denote by</p>
<div class="math notranslate nohighlight" id="equation-eq-4-31">
<span class="eqno">(61)<a class="headerlink" href="#equation-eq-4-31" title="Permalink to this equation">#</a></span>\[    T_i \sim \text{Geo}
\left(  \frac{{i \choose
      2}}{2N} \right).\]</div>
<p>The mean waiting time till any of pair within
our sample coalesces is</p>
<div class="math notranslate nohighlight" id="equation-eq-4-32">
<span class="eqno">(62)<a class="headerlink" href="#equation-eq-4-32" title="Permalink to this equation">#</a></span>\[    E( T_i) = \frac{2N}{{i \choose  2}}\]</div>
<p>which again
follows from the mean of a geometric random variable being
<span class="math notranslate nohighlight">\(\frac{1}{p}\)</span>.</p>
<p>After a pair of alleles first finds a common ancestral allele some
number of generations back in the past, we only have to keep track of
that common ancestral allele for the pair when looking further into the
past. In our example coalescent genealogy for our 3 alleles, shown in <a class="reference internal" href="#figure-4-18"><span class="std std-numref">Fig. 43</span></a>, we start by tracking the 3
lineages, then by chance the blue and purple coalesce in the four
generations back. Then we’re tracking just two lineages, the red lineage
and the ancestral lineage of the blue and purple alleles; then those two
coalesce and we’ve found our most recent common ancestor of our sample.
Another example with four tips is shown in <a class="reference internal" href="#figure-4-19"><span class="std std-numref">Fig. 44</span></a>; we’re track four lineages, then a pair
coalesce, then we tracking three lineages, then a pair coalesce, then
we’re tracking two lineages, then this final pair coalesce and we’ve
found the most recent common ancestor of our sample (fin, end scene).</p>
<p>More generally, when a pair of alleles in our sample of <span class="math notranslate nohighlight">\(i\)</span> alleles
coalesces, we then switch to having to follow <span class="math notranslate nohighlight">\(i-1\)</span> alleles back in
time. Then when a pair of these <span class="math notranslate nohighlight">\(i-1\)</span> alleles coalesce, we then only
have to follow <span class="math notranslate nohighlight">\(i-2\)</span> alleles back. This process continues until we
coalesce back to a sample of two, and from there to a single most recent
common ancestor (MRCA).</p>
<section id="simulating-a-coalescent-genealogy">
<h2>Simulating a coalescent genealogy<a class="headerlink" href="#simulating-a-coalescent-genealogy" title="Permalink to this headline">#</a></h2>
<p>To simulate a coalescent genealogy at a locus for a sample of <span class="math notranslate nohighlight">\(n\)</span>
alleles we therefore simply follow the following algorithm:</p>
<ol class="simple">
<li><p>Set <span class="math notranslate nohighlight">\(i=n\)</span>.</p></li>
<li><p>Simulate a random variable to be the time <span class="math notranslate nohighlight">\(T_i\)</span> to the next
coalescent event from <span class="math notranslate nohighlight">\(T_i \sim
  \text{Exp}\left(\frac{{i \choose
 2}}{2N} \right)\)</span></p></li>
<li><p>Choose a pair of alleles to coalesce at random from all possible
pairs.</p></li>
<li><p>Set <span class="math notranslate nohighlight">\(i=i-1\)</span></p></li>
<li><p>Continue looping steps 2-4 until <span class="math notranslate nohighlight">\(i=1\)</span>, i.e. the most recent common
ancestor of the sample is found.</p></li>
</ol>
<p>By following this algorithm we are generating realizations of the
genealogy of our sample.</p>
</section>
<section id="expected-properties-of-coalescent-genealogies-and-mutations">
<h2>Expected properties of coalescent genealogies and mutations<a class="headerlink" href="#expected-properties-of-coalescent-genealogies-and-mutations" title="Permalink to this headline">#</a></h2>
<figure class="align-left" id="figure-4-19">
<img alt="../../_images/Coal_w_muts.pdf" src="../../_images/Coal_w_muts.pdf" />
<figcaption>
<p><span class="caption-number">Fig. 44 </span><span class="caption-text">- A simple coalescent tree from a single coalescent simulation, tracing
the genealogy of 4 alleles with mutational changes marked with dashes
showing transitions away from the MRCA sequence (AGTTT) . The <span class="math notranslate nohighlight">\(T_{MRCA}\)</span>
is <span class="math notranslate nohighlight">\(T_4+T_3+T_2\)</span>. The total time in the tree is
<span class="math notranslate nohighlight">\(T_{tot}=4 T_4+3T_3 + 2T_2= 54\)</span> generations.</span><a class="headerlink" href="#figure-4-19" title="Permalink to this image">#</a></p>
</figcaption>
</figure>
<section id="the-expected-time-to-the-most-recent-common-ancestor">
<h3>The expected time to the most recent common ancestor<a class="headerlink" href="#the-expected-time-to-the-most-recent-common-ancestor" title="Permalink to this headline">#</a></h3>
<p>We will first consider the time to the most recent common ancestor of
the entire sample (<span class="math notranslate nohighlight">\(T_{MRCA}\)</span>). This is</p>
<div class="math notranslate nohighlight" id="equation-eq-4-34">
<span class="eqno">(63)<a class="headerlink" href="#equation-eq-4-34" title="Permalink to this equation">#</a></span>\[    T_{MRCA} = \sum_{i=n}^2 T_i\]</div>
<p>generations back, where we are summing from <span class="math notranslate nohighlight">\(i=n\)</span> alleles counting
backwards to <span class="math notranslate nohighlight">\(i=2\)</span> alleles (see <a class="reference internal" href="#figure-4-19"><span class="std std-numref">Fig. 44</span></a> for example). As our coalescent times for
different <span class="math notranslate nohighlight">\(i\)</span> are independent, the expected time to the most recent
common ancestor is</p>
<div class="math notranslate nohighlight" id="equation-eq-4-35">
<span class="eqno">(64)<a class="headerlink" href="#equation-eq-4-35" title="Permalink to this equation">#</a></span>\[    E(T_{MRCA}) = \sum_{i=n}^2 E(T_i) = \sum_{i=n}^2  2N/{i \choose
 2}\]</div>
<p>Using the fact that <span class="math notranslate nohighlight">\(\frac{1}{i(i-1)}=\frac{1}{i-1} - \frac{1}{i}\)</span>
and a bit of rearrangement, we can rewrite this as</p>
<div class="math notranslate nohighlight" id="equation-eq-4-36">
<span class="eqno">(65)<a class="headerlink" href="#equation-eq-4-36" title="Permalink to this equation">#</a></span>\[    E(T_{MRCA}) = 4N\left(1- \frac{1}{n} \right)\]</div>
<p>So the average <span class="math notranslate nohighlight">\(T_{MRCA}\)</span> scales linearly with population size <span class="math notranslate nohighlight">\(N\)</span>.
Interestingly, as we move to larger and larger samples (i.e. <span class="math notranslate nohighlight">\(n \gg 1\)</span>),
the average time to the most recent common ancestor converges on <span class="math notranslate nohighlight">\(4N\)</span>.
What’s happening here is that in large samples our lineages typically
coalesce rapidly at the start and very soon coalesce down to a much
smaller number of lineages.</p>
<div class="admonition-question-9 admonition" id="question-4-9">
<p class="admonition-title">Question 9</p>
<p>Assume an autosomal effective population of 10,000 individuals (roughly
the long-term human estimate) and a generation time of 30 years. What is
the expected time to the most recent common ancestor of a sample of 20
people? What is this time for a sample of 500 people?</p>
</div>
</section>
<section id="the-expected-total-time-in-a-genealogy-and-the-number-of-segregating-sites">
<h3>The expected total time in a genealogy and the number of segregating sites<a class="headerlink" href="#the-expected-total-time-in-a-genealogy-and-the-number-of-segregating-sites" title="Permalink to this headline">#</a></h3>
<p>Mutations fall on specific lineages of the coalescent genealogy and are
transmitted to all descendants of their lineage. Furthermore, under the
infinitely-many-sites assumption, each mutation creates a new
segregating site. The mutation process is a <em>Poisson process</em>, and the
longer a particular lineage, i.e. the more generations of meioses it
represents, the more mutations that can accumulate on it. The total
number of segregating sites in a sample is thus a function of the
<em>total</em> amount of time in the genealogy of the sample, or the sum of all
the branch lengths on the genealogical tree, <span class="math notranslate nohighlight">\(T_{tot}\)</span>. Our total amount
of time in the genealogy is</p>
<div class="math notranslate nohighlight" id="equation-eq-4-37">
<span class="eqno">(66)<a class="headerlink" href="#equation-eq-4-37" title="Permalink to this equation">#</a></span>\[    T_{tot} = \sum_{i=n}^2 iT_i\]</div>
<p>as when there are <span class="math notranslate nohighlight">\(i\)</span> lineages, each
contributes a time <span class="math notranslate nohighlight">\(T_i\)</span> to the total time (see <a class="reference internal" href="#figure-4-19"><span class="std std-numref">Fig. 44</span></a> for an example). Taking the expectation of
the total time in the genealogy,</p>
<div class="math notranslate nohighlight" id="equation-eq-4-38">
<span class="eqno">(67)<a class="headerlink" href="#equation-eq-4-38" title="Permalink to this equation">#</a></span>\[    E(T_{tot}) = \sum_{i=n}^2 i \frac{2N}{{i \choose
 2} } = \sum_{i=n}^2 \frac{4N}{i -1} =\sum_{i=n-1}^1 \frac{4N}{i}\]</div>
<p>we see that our expected total amount of time in the genealogy scales
linearly with our population size <span class="math notranslate nohighlight">\(N\)</span>. Our expected total amount of time
is also increasing with sample size <span class="math notranslate nohighlight">\(n\)</span>, but is doing so very slowly.
This again follows from the fact that in large samples, the initial
coalescence usually happens very rapidly, so that extra samples add
little to the total amount of time in the genealogical tree.<br />
We saw above that the number of mutational differences between a pair of
alleles that coalescence <span class="math notranslate nohighlight">\(T_2\)</span> generations ago was Poisson with a mean
of <span class="math notranslate nohighlight">\(2 \mu T_2\)</span>, where <span class="math notranslate nohighlight">\(2T_{2}\)</span> is the total branch length in this simple
2-sample genealogical tree. A mutation that occurs on any branch of our
genealogy will cause a segregating polymorphism in the sample (meeting
our infinitely-many-sites assumption). Thus, if the total time in the
genealogy is <span class="math notranslate nohighlight">\(T_{tot}\)</span>, there are <span class="math notranslate nohighlight">\(T_{tot}\)</span> generations for mutations.
So the total number of mutations segregating in our sample (<span class="math notranslate nohighlight">\(S\)</span>) is
Poisson with mean <span class="math notranslate nohighlight">\(\mu T_{tot}\)</span>. Thus the expected number of segregating
sites in a sample of size <span class="math notranslate nohighlight">\(n\)</span> is</p>
<div class="math notranslate nohighlight" id="equation-eq-4-39">
<span class="eqno">(68)<a class="headerlink" href="#equation-eq-4-39" title="Permalink to this equation">#</a></span>\[    E(S) = \mu \E(T_{tot}) = \sum_{i=n-1}^1 \frac{4N\mu }{i} = \theta
\sum_{i=n-1}^1 \frac{1}{i}\]</div>
<p>Note that this is
growing with the sample size <span class="math notranslate nohighlight">\(n\)</span>, albeit very slowly (roughly at the
rate of the <span class="math notranslate nohighlight">\(\log\)</span> of the sample size). We can use this formula to
derive another estimate of the population scaled mutation rate <span class="math notranslate nohighlight">\(\theta\)</span>,
by setting our observed number of segregating sites in a sample (<span class="math notranslate nohighlight">\(S\)</span>)
equal to this expectation. We’ll call this estimator
<span class="math notranslate nohighlight">\(\widehat{\theta}_W\)</span>:</p>
<div class="math notranslate nohighlight" id="equation-eq-4-40">
<span class="eqno">(69)<a class="headerlink" href="#equation-eq-4-40" title="Permalink to this equation">#</a></span>\[    \widehat{\theta}_W =\frac{ S}{\sum_{i=n-1}^1 \frac{1}{i}}\]</div>
<p>This estimator of <span class="math notranslate nohighlight">\(\theta\)</span> was devised by <span id="id1">Watterson [<a class="reference internal" href="../chapter-00-bibliography.html#id301" title="GA Watterson. On the number of segregating sites in genetical models without recombination. Theoretical population biology, 7(2):256–276, 1975.">1975</a>]</span>, hence the <span class="math notranslate nohighlight">\(W\)</span>.</p>
</section>
<section id="the-neutral-site-frequency-spectrum">
<h3>The neutral site-frequency spectrum<a class="headerlink" href="#the-neutral-site-frequency-spectrum" title="Permalink to this headline">#</a></h3>
<p>We can use our coalescent process to find the expected number of derived
alleles present <span class="math notranslate nohighlight">\(i\)</span> times out of a sample size <span class="math notranslate nohighlight">\(n\)</span>, e.g. how many
singletons (<span class="math notranslate nohighlight">\(i = 1\)</span>) do we expect to find in our sample? For example, in
<a class="reference internal" href="#figure-4-19"><span class="std std-numref">Fig. 44</span></a> in our sample of four sequences, there are
3 singletons and 2 doubletons. The number of sites with these different
allele frequencies depends on the lengths of specific genealogical
branches. A mutation that falls on a branch with <span class="math notranslate nohighlight">\(i\)</span> descendants will
create a derived allele with frequency <span class="math notranslate nohighlight">\(i\)</span>. For example, in our example
tree in <a class="reference internal" href="#figure-4-19"><span class="std std-numref">Fig. 44</span></a>, the total number of generations where a
mutation could arise and be a doubleton is <span class="math notranslate nohighlight">\(T_3+2T_2\)</span>, the total length
of the branch ancestral to just the orange and red allele <span class="math notranslate nohighlight">\((T_3+T_2)\)</span>
plus the branch ancestral to just the blue and purple allele <span class="math notranslate nohighlight">\((T_2)\)</span>.</p>
<p>To see how we could go about working this out, let’s start by
considering the simple coalescent tree, shown in <code class="xref std std-numref docutils literal notranslate"><span class="pre">figure-4.20</span></code>, for sample of <span class="math notranslate nohighlight">\(3\)</span> alleles drawn from a
population. Mutations that fall on the branches coloured in black will
be derived singletons, while mutations that fall along the orange branch
will be doubletons in the sample. The total number of generations where
a singleton mutation could arise is <span class="math notranslate nohighlight">\(3 T_3 + T_2\)</span>. Note that we only
count the time where there are two lineages <span class="math notranslate nohighlight">\((T_{2})\)</span> once. So our
expected number of singletons, using equation <a class="reference internal" href="#equation-eq-4-32">(62)</a>, is</p>
<div class="math notranslate nohighlight" id="equation-eq-4-41">
<span class="eqno">(70)<a class="headerlink" href="#equation-eq-4-41" title="Permalink to this equation">#</a></span>\[    E(S_i) = \mu \left( 3E(T_3) +  E(T_2) \right) = \mu \left( 3
  \frac{2N}{3}+ 2N \right) = \theta\]</div>
<p>By similar logic, the time where
doubletons could arise is <span class="math notranslate nohighlight">\(T_2\)</span> and our expected number of doubletons is
<span class="math notranslate nohighlight">\(E(S_i)
=\theta/2\)</span>. Thus, there are on average half as many doubletons as
singletons.</p>
<p>Extending this logic to larger samples might be doable, but is tedious
(I mean really tedious: for 10 alleles there are thousands of possible
tree shapes and the task quickly gets impossible even computationally).
A nice, relatively simple proof of the neutral site frequency spectrum
is given by <span id="id2">Hudson [<a class="reference internal" href="../chapter-00-bibliography.html#id231" title="Richard R. Hudson. A new proof of the expected frequency spectrum under the standard neutral model. PLOS ONE, 10(7):1-5, 07 2015. URL: https://doi.org/10.1371/journal.pone.0118087, doi:10.1371/journal.pone.0118087.">2015</a>]</span>, but we won’t give this here. The general form
is:</p>
<div class="math notranslate nohighlight" id="equation-eq-4-42">
<span class="eqno">(71)<a class="headerlink" href="#equation-eq-4-42" title="Permalink to this equation">#</a></span>\[    E(S_i) = \frac{\theta }{i}\]</div>
<p>i.e.
there are twice as many singletons as doubletons, three times as many
singletons as tripletons, and so on. The other thing that will be
helpful for us to know is that neutral alleles at intermediate frequency
tend to be old, and those that are rare in the sample are on average
young. We expect to see a lot more rare alleles in our sample than
common alleles.</p>
<div class="admonition-question-10 admonition" id="question-4-10">
<p class="admonition-title">Question 10</p>
<p>There are two possible tree shapes that could relate four samples. Draw
both of them and separately colour (or otherwise mark) the branches by
where singletons, doubletons, and tripleton derived alleles could arise.</p>
</div>
<p>We can also ask the probability of observing a derived allele
segregating at frequency <span class="math notranslate nohighlight">\(i/n\)</span> given that the site is polymorphic in our
sample of size <span class="math notranslate nohighlight">\(n\)</span> (i.e. given that <span class="math notranslate nohighlight">\(0&lt;i&lt;n\)</span> ). We can obtain this
probability by dividing the expected number of sites segregating for an
allele at frequency <span class="math notranslate nohighlight">\(i\)</span> by the expected number segregating at all of the
possible allele frequencies for polymorphisms in our sample</p>
<div class="math notranslate nohighlight" id="equation-eq-4-43">
<span class="eqno">(72)<a class="headerlink" href="#equation-eq-4-43" title="Permalink to this equation">#</a></span>\[    \begin{aligned}
    P(i |0&lt;i&lt;n) &amp;=\frac{E(S_i)}{\sum_{j=1}^{n-1} E(S_j)} = \frac{\frac{1}{i}}{\sum_{j=1}^{n-1} \frac{1}{j}}.\end{aligned}\]</div>
<p>We can interpret this probability as the fraction of polymorphic sites
we expect to find at a frequency <span class="math notranslate nohighlight">\(i/n\)</span>.</p>
</section>
<section id="tests-based-on-the-site-frequency-spectrum">
<h3>Tests based on the site frequency spectrum<a class="headerlink" href="#tests-based-on-the-site-frequency-spectrum" title="Permalink to this headline">#</a></h3>
<p>Population geneticists have proposed a variety of ways to test whether
an observed site frequency spectrum conforms to its neutral,
constant-size expectations. These tests are useful for detecting
population size changes using data across many loci, or for detecting
the signal of selection at individual loci. One of the first tests was
proposed by <span id="id3">Tajima [<a class="reference internal" href="../chapter-00-bibliography.html#id254" title="Fumio Tajima. Statistical method for testing the neutral mutation hypothesis by dna polymorphism. Genetics, 123(3):585–595, 1989.">1989</a>]</span>, and is called Tajima’s <span class="math notranslate nohighlight">\(D\)</span>. Tajima’s <span class="math notranslate nohighlight">\(D\)</span> is</p>
<div class="math notranslate nohighlight" id="equation-eq-4-44">
<span class="eqno">(73)<a class="headerlink" href="#equation-eq-4-44" title="Permalink to this equation">#</a></span>\[    D = \frac{\hat{\theta}_{\pi}-\hat{\theta}_{W}}{C}\]</div>
<p>where the numerator is the difference between the estimate of <span class="math notranslate nohighlight">\(\theta\)</span>
based on pairwise differences and that based on segregating sites. As
these two estimators both have expectation <span class="math notranslate nohighlight">\(\theta\)</span> under the neutral,
constant-size model, the expectation of <span class="math notranslate nohighlight">\(D\)</span> is zero. The denominator <span class="math notranslate nohighlight">\(C\)</span>
is a positive constant; it’s the square-root of an estimator of the
variance of this difference under the constant population size, neutral
model. This constant was chosen for <span class="math notranslate nohighlight">\(D\)</span> to have mean zero and variance
<span class="math notranslate nohighlight">\(1\)</span> under the null model, so we can test for departures from this simple
null model.</p>
<p>An excess of rare alleles compared to the constant-size, neutral model
will result in a negative Tajima’s <span class="math notranslate nohighlight">\(D\)</span>, because each additional rare
allele increases the number of segregating sites by <span class="math notranslate nohighlight">\(1\)</span>, but only has a
small effect on the number of pairwise differences between samples. In
contrast, a positive Tajima’s <span class="math notranslate nohighlight">\(D\)</span> reflects an excess of intermediate
frequency alleles relative to the constant-size, neutral expectation.
Alleles at intermediate-frequency increase pairwise diversity more per
segregating site than typical, thus increasing <span class="math notranslate nohighlight">\(\theta_{\pi}\)</span> more than
<span class="math notranslate nohighlight">\(\theta_{W}\)</span>. In the next section we’ll see how long-term changes in
population size systematically change the site frequency spectrum and so
are detectable by statistics such as Tajima’s <span class="math notranslate nohighlight">\(D\)</span>.</p>
</section>
</section>
<section id="demography-and-the-coalescent">
<h2>Demography and the coalescent<a class="headerlink" href="#demography-and-the-coalescent" title="Permalink to this headline">#</a></h2>
<p>We’ve already seen how changes in population size can change the rate at
which heterozygosity is lost from the population (see the discussion
around equation <a class="reference internal" href="chapter-04.2-loss-of-heterozygosity-due-to-drift.html#equation-eq-4-15">(47)</a>). If the population size in generation <span class="math notranslate nohighlight">\(i\)</span>
is <span class="math notranslate nohighlight">\(N_i\)</span>, the probability that a pair of lineages coalesce is
<span class="math notranslate nohighlight">\(\frac{1}{(2N_i)}\)</span>; this conforms to our intuition that if the
population size is small, the rate at which pairs of lineages find their
common ancestor is faster. We can potentially accommodate rapid random
fluctuations in population size by simply using the effective population
size <span class="math notranslate nohighlight">\(N_e\)</span> in place of <span class="math notranslate nohighlight">\(N\)</span>. However, longer-term, more systematic
changes in population size will distort the coalescent genealogies, and
hence patterns of diversity, in more systematic ways.</p>
<p>We can see how demography potentially distorts the observed frequency
spectrum away from the neutral expectation in a very large sample of
humans shown in <code class="xref std std-numref docutils literal notranslate"><span class="pre">figure-4.21</span></code>. For comparison, the neutral frequency
spectrum, equation <a class="reference internal" href="#equation-eq-4-42">(71)</a>, is shown as a red line. There are
vastly more rare alleles than expected under our neutral,
constant-size-size model, but the neutral prediction and reality agree
somewhat more for alleles that are more common.</p>
<p>Why is this? Well, these patterns are likely the result of the very
recent explosive growth in human populations. If the population has
grown rapidly, then the pairwise-coalescent rate in the past may be much
higher than the coalescent rate closer to the present. (see <code class="xref std std-numref docutils literal notranslate"><span class="pre">figure-4.22</span></code>).</p>
<p>One consequence of a recent population expansion is that there is much
less genetic diversity in the population than you’d predict using the
census population size. Humans are one example of this effect; there are
<span class="math notranslate nohighlight">\(7\)</span> billion of us alive today, but this is due to very rapid population
growth over the past thousand to tens of thousands of years. Our level
of genetic diversity is very much lower than you’d predict given our
census size, reflecting our much smaller ancestral population. A second
consequence of recent population expansion is that the deeper coalescent
branches are much more squished together in time compared to those in a
constant-sized population. Mutations on deeper branches are the source
of alleles at more intermediate frequencies, and so there are even fewer
intermediate-frequency alleles in growing populations. That’s why there
are so many rare alleles, especially singletons, in this large sample of
Europeans.</p>
<p>Another common demographic scenario is a population bottleneck. In a
bottleneck, the population size crashes dramatically, and subsequently
recovers. For example, our population may have had size
<span class="math notranslate nohighlight">\(N_{\textrm{Big}}\)</span> and crashed down to <span class="math notranslate nohighlight">\(N_{\textrm{Small}}\)</span>. One example
of a bottleneck is shown in <code class="xref std std-numref docutils literal notranslate"><span class="pre">figure-4.23</span></code>. Looking at a sample of lineages drawn from the population today, if the
bottleneck was somewhat recent (<span class="math notranslate nohighlight">\(\ll N_{\textrm{Big}}\)</span> generations in
the past) many of our lineages will not have coalesced before reaching
the bottleneck, moving backward in time. But during the bottleneck our
lineages coalesce at a much higher rate, such that many of our lineages
will coalesce if the bottleneck lasts long enough
(<span class="math notranslate nohighlight">\(\sim N_{\textrm{Small}}\)</span> generations). If the bottleneck is very
strong, then all of our lineages will coalesce during the bottleneck,
and the resulting site frequency spectrum may look very much like our
population growth model (i.e. an excess of rare alleles). However, if
some pairs of lineages escape coalescing during the bottleneck, they
will coalesce much more deeply in time (e.g. the blue and orange
ancestral lineages in <code class="xref std std-numref docutils literal notranslate"><span class="pre">figure-4.23</span></code>).</p>
<figure class="align-left" id="figure-4-24">
<img alt="../../_images/Mimulus_coalescent_times.pdf" src="../../_images/Mimulus_coalescent_times.pdf" />
<figcaption>
<p><span class="caption-number">Fig. 45 </span><span class="caption-text">- Diversity along a region of the Mimulus genome. Black dots give <span class="math notranslate nohighlight">\(\pi\)</span>
in 1kb windows between chromosomes sampled from two individuals, the red
line is a moving average (data from <span id="id4">Brandvain <em>et al.</em> [<a class="reference internal" href="../chapter-00-bibliography.html#id266" title="Yaniv Brandvain, Amanda M Kenney, Lex Flagel, Graham Coop, and Andrea L Sweigart. Speciation and introgression between mimulus nasutus and mimulus guttatus. PLoS Genetics, 10(6):e1004410, 2014.">2014</a>]</span>). Pairwise coalescent
times (<span class="math notranslate nohighlight">\(t\)</span>) estimated assuming <span class="math notranslate nohighlight">\(t= \frac{\pi}{2 \mu}\)</span> using
<span class="math notranslate nohighlight">\(\mu_{BP}=10^{-9}\)</span>.</span><a class="headerlink" href="#figure-4-24" title="Permalink to this image">#</a></p>
</figcaption>
</figure>
<p>An example of this is shown <a class="reference internal" href="#figure-4-24"><span class="std std-numref">Fig. 45</span></a>, data from <span id="id5">Brandvain <em>et al.</em> [<a class="reference internal" href="../chapter-00-bibliography.html#id266" title="Yaniv Brandvain, Amanda M Kenney, Lex Flagel, Graham Coop, and Andrea L Sweigart. Speciation and introgression between mimulus nasutus and mimulus guttatus. PLoS Genetics, 10(6):e1004410, 2014.">2014</a>]</span>. <em>Mimulus
nasutus</em> is a selfing species that arose recently from an out-crossing
progenitor <em>M. guttatus</em>, and experienced a strong bottleneck. <em>M.
guttatus</em> has very high levels of genetic diversity (<span class="math notranslate nohighlight">\(\pi=4\%\)</span> at
synonymous sites), but <em>M. nasutus</em> has lost much of this diversity
(<span class="math notranslate nohighlight">\(\pi =1\%\)</span>). Looking along the genome, between a pair of <em>M. guttatus</em>
chromosomes, levels of diversity are fairly uniformly high.</p>
<p>But in comparing two <em>M. nasutus</em> chromosomes, diversity is low because
the pair of lineages generally coalesce recently. Yet in a few places we
see levels of diversity comparable to <em>M. guttatus</em>; these regions
correspond to genomic sites where our pair of lineages fail to coalesce
during the bottleneck and subsequently coalesce much more deeply in the
ancestral <em>M. guttatus</em> population.</p>
<figure class="align-left" id="figure-4-26">
<img alt="../../_images/Wright_Tajima_D.pdf" src="../../_images/Wright_Tajima_D.pdf" />
<figcaption>
<p><span class="caption-number">Fig. 46 </span><span class="caption-text">- Data for polymorphism from Maize and Teosinite: 774 loci from
<span id="id6">Wright <em>et al.</em> [<a class="reference internal" href="../chapter-00-bibliography.html#id230" title="Stephen I. Wright, Irie Vroh Bi, Steve G. Schroeder, Masanori Yamasaki, John F. Doebley, Michael D. McMullen, and Brandon S. Gaut. The effects of artificial selection on the maize genome. Science, 308(5726):1310–1314, 2005. URL: http://science.sciencemag.org/content/308/5726/1310, arXiv:http://science.sciencemag.org/content/308/5726/1310.full.pdf, doi:10.1126/science.1107891.">2005</a>]</span>. <strong>Left)</strong> Genetic diversity levels in maize and and teosinte
samples at each of these loci. Note how diversity levels are lower in
maize than teosinte, i.e. most points are below the red <span class="math notranslate nohighlight">\(x=y\)</span> line.
<strong>Right)</strong> The distribution of Tajima’s D in maize and teosinte, see how
the maize distribution is shifted towards positive values.</span><a class="headerlink" href="#figure-4-26" title="Permalink to this image">#</a></p>
</figcaption>
</figure>
<p>Mutations that arise on deeper lineages will be at intermediate
frequency in our sample, and so mild bottlenecks can lead to an excess
of intermediate frequency alleles compared to the standard constant-size
model. This can skew Tajima’s D (see equation <a class="reference internal" href="#equation-eq-4-44">(73)</a>) towards positive values and away from its
expectation of zero. One example of this skew is shown in <a class="reference internal" href="#figure-4-26"><span class="std std-numref">Fig. 46</span></a>. Maize (<em>Zea mays</em> subsp. <em>mays</em>) was
domesticated from its wild progenitor teosinte (<em>Zea mays subsp.
parviglumis</em>) roughly ten thousand years ago. We can see how the
bottleneck associated with domestication has resulted in a loss of
genetic diversity in maize compared to teosinte, and the polymorphism
that remains is somewhat skewed towards intermediate frequencies
resulting in more positive values of Tajima’s D.</p>
<div class="admonition-question-11 admonition" id="question-4-11">
<p class="admonition-title">Question 11</p>
<p><span id="id7">Voight <em>et al.</em> [<a class="reference internal" href="../chapter-00-bibliography.html#id198" title="Benjamin F Voight, Alison M Adams, Linda A Frisse, Yudong Qian, Richard R Hudson, and Anna Di Rienzo. Interrogating multiple aspects of variation in a full resequencing data set to infer human population size changes. Proceedings of the National Academy of Sciences, 102(51):18508–18513, 2005.">2005</a>]</span> sequenced 40 autosomal regions from 15 diploid
samples of Hausa people from Yaounde, Cameroon. The average length of
locus they sequenced for each region was <span class="math notranslate nohighlight">\(2365\)</span>bp. They found that the
average number of segregating sites per locus was <span class="math notranslate nohighlight">\(S= 11.1\)</span> and the
average <span class="math notranslate nohighlight">\(\pi = 0.0011\)</span> per base over the loci. Is Tajima’s D positive or
negative? Is a demographic model with a bottleneck or growth more
consistent with this result?</p>
</div>
</section>
</section>

    <script type="text/x-thebe-config">
    {
        requestKernel: true,
        binderOptions: {
            repo: "binder-examples/jupyter-stacks-datascience",
            ref: "master",
        },
        codeMirrorConfig: {
            theme: "abcdef",
            mode: "python"
        },
        kernelOptions: {
            kernelName: "python3",
            path: "./chapters/chapter-04"
        },
        predefinedOutput: true
    }
    </script>
    <script>kernelName = 'python3'</script>

              </div>
              
            </main>
            <footer class="footer-article noprint">
                
    <!-- Previous / next buttons -->
<div class='prev-next-area'>
    <a class='left-prev' id="prev-link" href="chapter-04.3-the-coalescent-and-patterns-of-neutral-diversity.html" title="previous page">
        <i class="fas fa-angle-left"></i>
        <div class="prev-next-info">
            <p class="prev-next-subtitle">previous</p>
            <p class="prev-next-title">The Coalescent and patterns of neutral diversity</p>
        </div>
    </a>
    <a class='right-next' id="next-link" href="chapter-04.5-chapter-summary.html" title="next page">
    <div class="prev-next-info">
        <p class="prev-next-subtitle">next</p>
        <p class="prev-next-title">Chapter Summary</p>
    </div>
    <i class="fas fa-angle-right"></i>
    </a>
</div>
            </footer>
        </div>
    </div>
    <div class="footer-content row">
        <footer class="col footer"><p>
  
    By Graham Coop<br/>
  
      &copy; Copyright 2022.<br/>
</p>
        </footer>
    </div>
    
</div>


      </div>
    </div>
  
  <!-- Scripts loaded after <body> so the DOM is not blocked -->
  <script src="../../_static/scripts/pydata-sphinx-theme.js?digest=1999514e3f237ded88cf"></script>


  </body>
</html>