
<!DOCTYPE html>

<html>
  <head>
    <meta charset="utf-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" /><meta name="generator" content="Docutils 0.17.1: http://docutils.sourceforge.net/" />

    <title>Allele Sharing Among Related Individuals And Identity By Descent &#8212; Population And Quantitative Genetics</title>
    
  <!-- Loaded before other Sphinx assets -->
  <link href="../../_static/styles/theme.css?digest=1999514e3f237ded88cf" rel="stylesheet">
<link href="../../_static/styles/pydata-sphinx-theme.css?digest=1999514e3f237ded88cf" rel="stylesheet">

    
  <link rel="stylesheet"
    href="../../_static/vendor/fontawesome/5.13.0/css/all.min.css">
  <link rel="preload" as="font" type="font/woff2" crossorigin
    href="../../_static/vendor/fontawesome/5.13.0/webfonts/fa-solid-900.woff2">
  <link rel="preload" as="font" type="font/woff2" crossorigin
    href="../../_static/vendor/fontawesome/5.13.0/webfonts/fa-brands-400.woff2">

    <link rel="stylesheet" type="text/css" href="../../_static/pygments.css" />
    <link rel="stylesheet" href="../../_static/styles/sphinx-book-theme.css?digest=5115cc725059bd94278eecd172e13a965bf8f5a9" type="text/css" />
    <link rel="stylesheet" type="text/css" href="../../_static/togglebutton.css" />
    <link rel="stylesheet" type="text/css" href="../../_static/copybutton.css" />
    <link rel="stylesheet" type="text/css" href="../../_static/mystnb.css" />
    <link rel="stylesheet" type="text/css" href="../../_static/sphinx-thebe.css" />
    <link rel="stylesheet" type="text/css" href="../../_static/design-style.b7bb847fb20b106c3d81b95245e65545.min.css" />
    
  <!-- Pre-loaded scripts that we'll load fully later -->
  <link rel="preload" as="script" href="../../_static/scripts/pydata-sphinx-theme.js?digest=1999514e3f237ded88cf">

    <script data-url_root="../../" id="documentation_options" src="../../_static/documentation_options.js"></script>
    <script src="../../_static/jquery.js"></script>
    <script src="../../_static/underscore.js"></script>
    <script src="../../_static/doctools.js"></script>
    <script src="../../_static/clipboard.min.js"></script>
    <script src="../../_static/copybutton.js"></script>
    <script src="../../_static/scripts/sphinx-book-theme.js?digest=9c920249402e914e316237a7dbc6769907cce411"></script>
    <script>let toggleHintShow = 'Click to show';</script>
    <script>let toggleHintHide = 'Click to hide';</script>
    <script>let toggleOpenOnPrint = 'true';</script>
    <script src="../../_static/togglebutton.js"></script>
    <script>var togglebuttonSelector = '.toggle, .admonition.dropdown, .tag_hide_input div.cell_input, .tag_hide-input div.cell_input, .tag_hide_output div.cell_output, .tag_hide-output div.cell_output, .tag_hide_cell.cell, .tag_hide-cell.cell';</script>
    <script src="../../_static/design-tabs.js"></script>
    <script>const THEBE_JS_URL = "https://unpkg.com/thebe@0.8.2/lib/index.js"
const thebe_selector = ".thebe,.cell"
const thebe_selector_input = "pre"
const thebe_selector_output = ".output, .cell_output"
</script>
    <script async="async" src="../../_static/sphinx-thebe.js"></script>
    <script>window.MathJax = {"options": {"processHtmlClass": "tex2jax_process|mathjax_process|math|output_area"}}</script>
    <script defer="defer" src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-mml-chtml.js"></script>
    <link rel="index" title="Index" href="../../genindex.html" />
    <link rel="search" title="Search" href="../../search.html" />
    <link rel="next" title="Chapter Summary" href="chapter-02.4-chapter-summary.html" />
    <link rel="prev" title="Allele Frequencies" href="chapter-02.2-allele-frequencies.html" />
    <meta name="viewport" content="width=device-width, initial-scale=1" />
    <meta name="docsearch:language" content="None">
    

    <!-- Google Analytics -->
    
  </head>
  <body data-spy="scroll" data-target="#bd-toc-nav" data-offset="60">
<!-- Checkboxes to toggle the left sidebar -->
<input type="checkbox" class="sidebar-toggle" name="__navigation" id="__navigation" aria-label="Toggle navigation sidebar">
<label class="overlay overlay-navbar" for="__navigation">
    <div class="visually-hidden">Toggle navigation sidebar</div>
</label>
<!-- Checkboxes to toggle the in-page toc -->
<input type="checkbox" class="sidebar-toggle" name="__page-toc" id="__page-toc" aria-label="Toggle in-page Table of Contents">
<label class="overlay overlay-pagetoc" for="__page-toc">
    <div class="visually-hidden">Toggle in-page Table of Contents</div>
</label>
<!-- Headers at the top -->
<div class="announcement header-item noprint"></div>
<div class="header header-item noprint"></div>

    
    <div class="container-fluid" id="banner"></div>

    

    <div class="container-xl">
      <div class="row">
          
<!-- Sidebar -->
<div class="bd-sidebar noprint" id="site-navigation">
    <div class="bd-sidebar__content">
        <div class="bd-sidebar__top"><div class="navbar-brand-box">
    <a class="navbar-brand text-wrap" href="../../index.html">
      
        <!-- `logo` is deprecated in Sphinx 4.0, so remove this when we stop supporting 3 -->
        
      
      
      <img src="../../_static/logo.png" class="logo" alt="logo">
      
      
      <h1 class="site-logo" id="site-title">Population And Quantitative Genetics</h1>
      
    </a>
</div><form class="bd-search d-flex align-items-center" action="../../search.html" method="get">
  <i class="icon fas fa-search"></i>
  <input type="search" class="form-control" name="q" id="search-input" placeholder="Search this book..." aria-label="Search this book..." autocomplete="off" >
</form><nav class="bd-links" id="bd-docs-nav" aria-label="Main">
    <div class="bd-toc-item active">
        
        <ul class="nav bd-sidenav bd-sidenav__home-link">
            <li class="toctree-l1">
                <a class="reference internal" href="../chapter-01-introduction.html">
                    Introduction
                </a>
            </li>
        </ul>
        <ul class="current nav bd-sidenav">
 <li class="toctree-l1 current active has-children">
  <a class="reference internal" href="chapter-02.1-allele-and-genotype-frequencies.html">
   Allele And Genotype Frequencies
  </a>
  <input checked="" class="toctree-checkbox" id="toctree-checkbox-1" name="toctree-checkbox-1" type="checkbox"/>
  <label for="toctree-checkbox-1">
   <i class="fas fa-chevron-down">
   </i>
  </label>
  <ul class="current">
   <li class="toctree-l2">
    <a class="reference internal" href="chapter-02.2-allele-frequencies.html">
     Allele Frequencies
    </a>
   </li>
   <li class="toctree-l2 current active">
    <a class="current reference internal" href="#">
     Allele Sharing Among Related Individuals And Identity By Descent
    </a>
   </li>
   <li class="toctree-l2">
    <a class="reference internal" href="chapter-02.4-chapter-summary.html">
     Chapter Summary
    </a>
   </li>
   <li class="toctree-l2">
    <a class="reference internal" href="chapter-02.5-further-questions.html">
     Further Questions
    </a>
   </li>
  </ul>
 </li>
 <li class="toctree-l1 has-children">
  <a class="reference internal" href="../chapter-03/chapter-03.1-population-structure-and-correlations-among-loci.html">
   Population Structure and Correlations Among Loci
  </a>
  <input class="toctree-checkbox" id="toctree-checkbox-2" name="toctree-checkbox-2" type="checkbox"/>
  <label for="toctree-checkbox-2">
   <i class="fas fa-chevron-down">
   </i>
  </label>
  <ul>
   <li class="toctree-l2">
    <a class="reference internal" href="../chapter-03/chapter-03.2-chapter-summary.html">
     Chapter Summary
    </a>
   </li>
   <li class="toctree-l2">
    <a class="reference internal" href="../chapter-03/chapter-03.3-further-questions.html">
     Further Questions
    </a>
   </li>
  </ul>
 </li>
 <li class="toctree-l1 has-children">
  <a class="reference internal" href="../chapter-04/chapter-04.1-genetic-drift-and-neutral-diversity.html">
   Genetic Drift and Neutral Diversity
  </a>
  <input class="toctree-checkbox" id="toctree-checkbox-3" name="toctree-checkbox-3" type="checkbox"/>
  <label for="toctree-checkbox-3">
   <i class="fas fa-chevron-down">
   </i>
  </label>
  <ul>
   <li class="toctree-l2">
    <a class="reference internal" href="../chapter-04/chapter-04.2-loss-of-heterozygosity-due-to-drift.html">
     Loss of heterozygosity due to drift
    </a>
   </li>
  </ul>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../chapter-00-bibliography.html">
   Bibliography
  </a>
 </li>
</ul>

    </div>
</nav></div>
        <div class="bd-sidebar__bottom">
             <!-- To handle the deprecated key -->
            
            <div class="navbar_extra_footer">
            Powered by <a href="https://jupyterbook.org">Jupyter Book</a>
            </div>
            
        </div>
    </div>
    <div id="rtd-footer-container"></div>
</div>


          


          
<!-- A tiny helper pixel to detect if we've scrolled -->
<div class="sbt-scroll-pixel-helper"></div>
<!-- Main content -->
<div class="col py-0 content-container">
    
    <div class="header-article row sticky-top noprint">
        



<div class="col py-1 d-flex header-article-main">
    <div class="header-article__left">
        
        <label for="__navigation"
  class="headerbtn"
  data-toggle="tooltip"
data-placement="right"
title="Toggle navigation"
>
  

<span class="headerbtn__icon-container">
  <i class="fas fa-bars"></i>
  </span>

</label>

        
    </div>
    <div class="header-article__right">
<button onclick="toggleFullScreen()"
  class="headerbtn"
  data-toggle="tooltip"
data-placement="bottom"
title="Fullscreen mode"
>
  

<span class="headerbtn__icon-container">
  <i class="fas fa-expand"></i>
  </span>

</button>

<div class="menu-dropdown menu-dropdown-repository-buttons">
  <button class="headerbtn menu-dropdown__trigger"
      aria-label="Source repositories">
      <i class="fab fa-github"></i>
  </button>
  <div class="menu-dropdown__content">
    <ul>
      <li>
        <a href="https://github.com/executablebooks/jupyter-book"
   class="headerbtn"
   data-toggle="tooltip"
data-placement="left"
title="Source repository"
>
  

<span class="headerbtn__icon-container">
  <i class="fab fa-github"></i>
  </span>
<span class="headerbtn__text-container">repository</span>
</a>

      </li>
      
      <li>
        <a href="https://github.com/executablebooks/jupyter-book/issues/new?title=Issue%20on%20page%20%2Fchapters/chapter-02/chapter-02.3-allele-sharing-among-related-individuals-and-identity-by-descent.html&body=Your%20issue%20content%20here."
   class="headerbtn"
   data-toggle="tooltip"
data-placement="left"
title="Open an issue"
>
  

<span class="headerbtn__icon-container">
  <i class="fas fa-lightbulb"></i>
  </span>
<span class="headerbtn__text-container">open issue</span>
</a>

      </li>
      
    </ul>
  </div>
</div>

<div class="menu-dropdown menu-dropdown-download-buttons">
  <button class="headerbtn menu-dropdown__trigger"
      aria-label="Download this page">
      <i class="fas fa-download"></i>
  </button>
  <div class="menu-dropdown__content">
    <ul>
      <li>
        <a href="../../_sources/chapters/chapter-02/chapter-02.3-allele-sharing-among-related-individuals-and-identity-by-descent.md"
   class="headerbtn"
   data-toggle="tooltip"
data-placement="left"
title="Download source file"
>
  

<span class="headerbtn__icon-container">
  <i class="fas fa-file"></i>
  </span>
<span class="headerbtn__text-container">.md</span>
</a>

      </li>
      
      <li>
        
<button onclick="printPdf(this)"
  class="headerbtn"
  data-toggle="tooltip"
data-placement="left"
title="Print to PDF"
>
  

<span class="headerbtn__icon-container">
  <i class="fas fa-file-pdf"></i>
  </span>
<span class="headerbtn__text-container">.pdf</span>
</button>

      </li>
      
    </ul>
  </div>
</div>
<label for="__page-toc"
  class="headerbtn headerbtn-page-toc"
  
>
  

<span class="headerbtn__icon-container">
  <i class="fas fa-list"></i>
  </span>

</label>

    </div>
</div>

<!-- Table of contents -->
<div class="col-md-3 bd-toc show noprint">
    <div class="tocsection onthispage pt-5 pb-3">
        <i class="fas fa-list"></i> Contents
    </div>
    <nav id="bd-toc-nav" aria-label="Page">
        <ul class="visible nav section-nav flex-column">
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#genotypic-sharing-between-pairs-of-individuals">
   Genotypic sharing between pairs of individuals
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#sharing-of-genomic-blocks-among-relatives">
   Sharing of genomic blocks among relatives.
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#inbreeding">
   Inbreeding
  </a>
  <ul class="nav section-nav flex-column">
   <li class="toc-h3 nav-item toc-entry">
    <a class="reference internal nav-link" href="#multiple-inbreeding-loops-in-a-pedigree">
     Multiple inbreeding loops in a pedigree.
    </a>
   </li>
  </ul>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#calculating-inbreeding-coefficients-from-genetic-data">
   Calculating inbreeding coefficients from genetic data
  </a>
  <ul class="nav section-nav flex-column">
   <li class="toc-h3 nav-item toc-entry">
    <a class="reference internal nav-link" href="#genomic-blocks-of-homozygosity-due-to-inbreeding">
     Genomic blocks of homozygosity due to inbreeding.
    </a>
   </li>
  </ul>
 </li>
</ul>

    </nav>
</div>
    </div>
    <div class="article row">
        <div class="col pl-md-3 pl-lg-5 content-container">
            <!-- Table of contents that is only displayed when printing the page -->
            <div id="jb-print-docs-body" class="onlyprint">
                <h1>Allele Sharing Among Related Individuals And Identity By Descent</h1>
                <!-- Table of contents -->
                <div id="print-main-content">
                    <div id="jb-print-toc">
                        
                        <div>
                            <h2> Contents </h2>
                        </div>
                        <nav aria-label="Page">
                            <ul class="visible nav section-nav flex-column">
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#genotypic-sharing-between-pairs-of-individuals">
   Genotypic sharing between pairs of individuals
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#sharing-of-genomic-blocks-among-relatives">
   Sharing of genomic blocks among relatives.
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#inbreeding">
   Inbreeding
  </a>
  <ul class="nav section-nav flex-column">
   <li class="toc-h3 nav-item toc-entry">
    <a class="reference internal nav-link" href="#multiple-inbreeding-loops-in-a-pedigree">
     Multiple inbreeding loops in a pedigree.
    </a>
   </li>
  </ul>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#calculating-inbreeding-coefficients-from-genetic-data">
   Calculating inbreeding coefficients from genetic data
  </a>
  <ul class="nav section-nav flex-column">
   <li class="toc-h3 nav-item toc-entry">
    <a class="reference internal nav-link" href="#genomic-blocks-of-homozygosity-due-to-inbreeding">
     Genomic blocks of homozygosity due to inbreeding.
    </a>
   </li>
  </ul>
 </li>
</ul>

                        </nav>
                    </div>
                </div>
            </div>
            <main id="main-content" role="main">
                
              <div>
                
  <section class="tex2jax_ignore mathjax_ignore" id="allele-sharing-among-related-individuals-and-identity-by-descent">
<h1>Allele Sharing Among Related Individuals And Identity By Descent<a class="headerlink" href="#allele-sharing-among-related-individuals-and-identity-by-descent" title="Permalink to this headline">#</a></h1>
<p>All of the individuals in a population are related to each other by a giant pedigree (family tree). For most pairs of individuals in a population these relationships are very distant (e.g. distant cousins), while some individuals will be more closely related (e.g. sibling/first cousins). All individuals are related to one another by varying levels of relatedness, or <em>kinship</em>. Related individuals can share alleles that have both descended from the shared common ancestor. To be shared, these alleles must be inherited through all meioses connecting the two individuals (e.g. surviving the <span class="math notranslate nohighlight">\(\frac{1}{2}\)</span> probability of segregation each meiosis). As closer relatives are separated by fewer meioses, closer relatives share more alleles. In <code class="xref std std-numref docutils literal notranslate"><span class="pre">figure-2.10</span></code>, we show the sharing of chromosomal regions between two cousins. As we’ll see, many population and quantitative genetic concepts rely on how closely related individuals are, and thus we need some way to quantify the degree of kinship among individuals.</p>
<figure class="align-left" id="figure-2-10">
<img alt="../../_images/Cousins_IBD_chromo_cartoon.png" src="../../_images/Cousins_IBD_chromo_cartoon.png" />
<figcaption>
<p><span class="caption-number">Fig. 10 </span><span class="caption-text">- First cousins sharing a stretch of chromosome identical by descent. The different grandparental diploid chromosomes are coloured so we can track them and recombinations between them across the generations. Notice that the identity by descent between the cousins persists for a long stretch of chromosome due to the limited number of generations for recombination. The squares represent males and circles females.</span><a class="headerlink" href="#figure-2-10" title="Permalink to this image">#</a></p>
</figcaption>
</figure>
<aside class="margin sidebar">
<p class="sidebar-title"></p>
<p><span id="id1">[<a class="reference internal" href="../chapter-00-bibliography.html#id307" title="Charles William Cotterman. A calculus for statistico-genetics. PhD thesis, The Ohio State University, 1940.">Cotterman, 1940</a>, <a class="reference internal" href="../chapter-00-bibliography.html#id308">Malécot, 1948</a>]</span></p>
</aside>
<p>We will define two alleles to be identical by descent (IBD) if they are identical due to transmission from a common ancestor in the past few generations. For the moment, we ignore mutation, and we will be more precise about what we mean by ‘past few generations’ later on. For example, parent and child share exactly one allele identical by descent at a locus, assuming that the two parents of the child are randomly mated individuals from the population. In <a class="reference internal" href="#figure-2-16"><span class="std std-numref">Fig. 16</span></a>, I show a pedigree demonstrating
some configurations of IBD.</p>
<aside class="margin sidebar">
<p class="sidebar-title"></p>
<figure class="align-left" id="figure-2-11">
<img alt="../../_images/IBD_0_1_2.pdf" src="../../_images/IBD_0_1_2.pdf" />
<figcaption>
<p><span class="caption-number">Fig. 11 </span><span class="caption-text">- A pair of diploid individ- uals (i and j) sharing 0, 1, or 2 alleles IBD where lines show the sharing of alleles by descent (e.g. from a shared ancestor). Here we’ll focus on IBD of outbred individuals. Dealing with sharing between inbred individuals requires 6 more identity-by-descent r coefficients, which honestly makes my head spin.</span><a class="headerlink" href="#figure-2-11" title="Permalink to this image">#</a></p>
</figcaption>
</figure>
</aside>
<p>One summary of how related two individuals (let’s call them <span class="math notranslate nohighlight">\(i\)</span> and <span class="math notranslate nohighlight">\(j\)</span>) are is the probability that our pair of individuals share 0, 1, or 2 alleles identical by descent (see <code class="xref std std-numref docutils literal notranslate"><span class="pre">figure-2.11</span></code>). We denote these identity-by-descent probabilities by <span class="math notranslate nohighlight">\(r_0\)</span>, <span class="math notranslate nohighlight">\(r_1\)</span>, and <span class="math notranslate nohighlight">\(r_2\)</span> respectively. See <a class="reference internal" href="#table-2-4"><span class="std std-numref">Table 4</span></a> for some examples. We can also interpret these probabilities as genome-wide averages. For example, on average, at a quarter of all their autosomal loci full-sibs share zero alleles identical by descent.</p>
<p>One summary of relatedness that will be important is the probability that two alleles (I &amp; J) picked at random, one from each of the two different individuals <span class="math notranslate nohighlight">\(i\)</span> and <span class="math notranslate nohighlight">\(j\)</span>, are identical by descent (<span class="math notranslate nohighlight">\(P(\text{I&amp;J IBD})\)</span>). We call this quantity the <em>coefficient of kinship</em> of individuals <span class="math notranslate nohighlight">\(i\)</span> and <span class="math notranslate nohighlight">\(j\)</span>, and denote it by <span class="math notranslate nohighlight">\(F_{ij}\)</span>. It is calculated as</p>
<div class="math notranslate nohighlight" id="equation-eq-2-3">
<span class="eqno">(3)<a class="headerlink" href="#equation-eq-2-3" title="Permalink to this equation">#</a></span>\[F_{ij} = P(\text{I&amp;J IBD})\]</div>
<div class="math notranslate nohighlight" id="equation-eq-2-4">
<span class="eqno">(4)<a class="headerlink" href="#equation-eq-2-4" title="Permalink to this equation">#</a></span>\[ \begin{align}\begin{aligned}F_{ij} = P(\text{I&amp;J IBD | i&amp;j 0 IBD})P(\text{i&amp;j 0 IBD}) \\+ P(\text{I&amp;J IBD | i&amp;j 1 IBD})P(\text{i&amp;j 1 IBD}) \\+ P(\text{I&amp;J IBD | i&amp;j 2 IBD})P(\text{i&amp;j 2 IBD})\end{aligned}\end{align} \]</div>
<div class="math notranslate nohighlight" id="equation-eq-2-5">
<span class="eqno">(5)<a class="headerlink" href="#equation-eq-2-5" title="Permalink to this equation">#</a></span>\[F_{ij} = 0 \times r_0 + \frac{1}{4} r_1  + \frac{1}{2} r_2.\]</div>
<p>In the above step, Equation <a class="reference internal" href="#equation-eq-2-4">(4)</a>, we’re summing the conditional probability of alleles <span class="math notranslate nohighlight">\(I\)</span> &amp; <span class="math notranslate nohighlight">\(J\)</span> being IBD over whether our individuals <span class="math notranslate nohighlight">\(i\)</span> &amp; <span class="math notranslate nohighlight">\(j\)</span> share <span class="math notranslate nohighlight">\(0\)</span>, <span class="math notranslate nohighlight">\(1\)</span>, or <span class="math notranslate nohighlight">\(2\)</span> alleles IBD, an example of using the Law of Total Probability (see Appendix Equation <code class="xref eq docutils literal notranslate"><span class="pre">eq-A.12</span></code>). We’ve then, in Equation <a class="reference internal" href="#equation-eq-2-5">(5)</a>, used the fact that we can calculate our condition probabilities of I &amp; J being IBD using the rules of Mendelian transmision. Consider the probability <span class="math notranslate nohighlight">\(P(\text{I&amp;J IBD} ~|~ \text{i&amp;j  1 IBD})\)</span>, i.e. that our pair of alleles (<span class="math notranslate nohighlight">\(I\)</span> &amp; <span class="math notranslate nohighlight">\(J\)</span>) drawn from individuals <span class="math notranslate nohighlight">\(i\)</span> and <span class="math notranslate nohighlight">\(j\)</span> are IBD given that <span class="math notranslate nohighlight">\(i\)</span> and <span class="math notranslate nohighlight">\(j\)</span> share one allele IBD, this is a <span class="math notranslate nohighlight">\(\frac{1}{4}\)</span> as we need to draw the allele that is IBD from both <span class="math notranslate nohighlight">\(i\)</span> and <span class="math notranslate nohighlight">\(j\)</span>, i.e. drawing both black alleles in the middle panel of <code class="xref std std-numref docutils literal notranslate"><span class="pre">figure-2.11</span></code>, which happens with probability <span class="math notranslate nohighlight">\(\frac{1}{2} \times \frac{1}{2}\)</span>. The coefficient of kinship will appear multiple times, in both our discussion of inbreeding and in the context of phenotypic resemblance between relatives.</p>
<table class="colwidths-auto table" id="table-2-4">
<caption><span class="caption-number">Table 4 </span><span class="caption-text">- Probability that two individuals (<span class="math notranslate nohighlight">\(i\)</span> and <span class="math notranslate nohighlight">\(j\)</span>) of a given relationship share 0, 1, or 2 alleles identical by descent on the autosomes. *Assuming that our individuals are outbred and that this the only close relationship the pair shares.</span><a class="headerlink" href="#table-2-4" title="Permalink to this table">#</a></caption>
<thead>
<tr class="row-odd"><th class="text-align:center head"><p>Relationship (i,j)*</p></th>
<th class="text-align:center head"><p>P(i&amp;j 0 IBD)</p></th>
<th class="text-align:center head"><p>P(i&amp;j 1 IBD)</p></th>
<th class="text-align:center head"><p>P(i&amp;j 2 IBD)</p></th>
<th class="text-align:center head"><p>P(I&amp;J IBD)</p></th>
</tr>
</thead>
<tbody>
<tr class="row-even"><td class="text-align:center"><p>Relationship (i,j)*</p></td>
<td class="text-align:center"><p><span class="math notranslate nohighlight">\(r_{0}\)</span></p></td>
<td class="text-align:center"><p><span class="math notranslate nohighlight">\(r_{1}\)</span></p></td>
<td class="text-align:center"><p><span class="math notranslate nohighlight">\(r_{2}\)</span></p></td>
<td class="text-align:center"><p><span class="math notranslate nohighlight">\(F_{ij}\)</span></p></td>
</tr>
<tr class="row-odd"><td class="text-align:center"><p>parent-child</p></td>
<td class="text-align:center"><p>0</p></td>
<td class="text-align:center"><p>1</p></td>
<td class="text-align:center"><p>0</p></td>
<td class="text-align:center"><p><span class="math notranslate nohighlight">\(\frac{1}{4}\)</span></p></td>
</tr>
<tr class="row-even"><td class="text-align:center"><p>full siblings</p></td>
<td class="text-align:center"><p><span class="math notranslate nohighlight">\(\frac{1}{4}\)</span></p></td>
<td class="text-align:center"><p><span class="math notranslate nohighlight">\(\frac{1}{2}\)</span></p></td>
<td class="text-align:center"><p><span class="math notranslate nohighlight">\(\frac{1}{4}\)</span></p></td>
<td class="text-align:center"><p><span class="math notranslate nohighlight">\(\frac{1}{4}\)</span></p></td>
</tr>
<tr class="row-odd"><td class="text-align:center"><p>Monozygotic twins</p></td>
<td class="text-align:center"><p>0</p></td>
<td class="text-align:center"><p>0</p></td>
<td class="text-align:center"><p>1</p></td>
<td class="text-align:center"><p><span class="math notranslate nohighlight">\(\frac{1}{2}\)</span></p></td>
</tr>
<tr class="row-even"><td class="text-align:center"><p>1<sup>st</sup> cousins</p></td>
<td class="text-align:center"><p><span class="math notranslate nohighlight">\(\frac{3}{4}\)</span></p></td>
<td class="text-align:center"><p><span class="math notranslate nohighlight">\(\frac{1}{4}\)</span></p></td>
<td class="text-align:center"><p>0</p></td>
<td class="text-align:center"><p><span class="math notranslate nohighlight">\(\frac{1}{16}\)</span></p></td>
</tr>
</tbody>
</table>
<div class="admonition-question-6 admonition" id="question-2-6">
<p class="admonition-title">Question 6</p>
<p>What are <span class="math notranslate nohighlight">\(r_0\)</span>, <span class="math notranslate nohighlight">\(r_1\)</span>, and <span class="math notranslate nohighlight">\(r_2\)</span> for <span class="math notranslate nohighlight">\(\frac{1}{2}\)</span> sibs? (<span class="math notranslate nohighlight">\(\frac{1}{2}\)</span> sibs share one parent but not the other).</p>
</div>
<div class="admonition-question-7 admonition" id="question-2-7">
<p class="admonition-title">Question 7</p>
<p>Explain in words why <span class="math notranslate nohighlight">\(P(\text{I&amp;J IBD} ~|~ \text{i&amp;j  2 IBD}) = \frac{1}{2}\)</span>.</p>
</div>
<section id="genotypic-sharing-between-pairs-of-individuals">
<h2>Genotypic sharing between pairs of individuals<a class="headerlink" href="#genotypic-sharing-between-pairs-of-individuals" title="Permalink to this headline">#</a></h2>
<p>Our <span class="math notranslate nohighlight">\(r\)</span> coefficients are going to have various uses. For example, they allow us to calculate the probability of the genotypes of a pair of relatives. Consider a biallelic locus where allele <span class="math notranslate nohighlight">\(A_1\)</span> is at frequency <span class="math notranslate nohighlight">\(p\)</span>, and two individuals who have IBD allele sharing probabilities <span class="math notranslate nohighlight">\(r_0\)</span>, <span class="math notranslate nohighlight">\(r_1\)</span>, <span class="math notranslate nohighlight">\(r_2\)</span>. What is the overall probability that these two individuals are both homozygous for allele 1? Well that’s</p>
<div class="math notranslate nohighlight" id="equation-eq-2-6">
<span class="eqno">(6)<a class="headerlink" href="#equation-eq-2-6" title="Permalink to this equation">#</a></span>\[ \begin{align}\begin{aligned}P(\text{both }A_{1}A_{1}) = P(\text{both }A_{1}A_{1}\text{ | 0 alleles IBD})P(\text{0 alleles IBD})\\+ P(\text{both }A_{1}A_{1}\text{ | 1 alleles IBD})P(\text{1 alleles IBD})\\+ P(\text{both }A_{1}A_{1}\text{ | 2 alleles IBD})P(\text{2 alleles IBD})\end{aligned}\end{align} \]</div>
<p>Or, in our <span class="math notranslate nohighlight">\(r_0\)</span>, <span class="math notranslate nohighlight">\(r_1\)</span>, <span class="math notranslate nohighlight">\(r_2\)</span> notation:</p>
<div class="math notranslate nohighlight" id="equation-eq-2-7">
<span class="eqno">(7)<a class="headerlink" href="#equation-eq-2-7" title="Permalink to this equation">#</a></span>\[ \begin{align}\begin{aligned}P(\text{both }A_{1}A_{1}) = P(\text{both }A_{1}A_{1}\text{ | 0 alleles IBD})r_{0}\\+ P(\text{both }A_{1}A_{1}\text{ | 1 alleles IBD})r_{1}\\+ P(\text{both }A_{1}A_{1}\text{ | 2 alleles IBD})r_{2}\end{aligned}\end{align} \]</div>
<p>If our pair of relatives share <span class="math notranslate nohighlight">\(0\)</span> alleles IBD, then the probability that they are both homozygous is <span class="math notranslate nohighlight">\(P(\textrm{both } A_1 A_1 | \text{0 alleles IBD}) =p^2 \times p^2\)</span>, as all four alleles represent independent draws from the population. If they share <span class="math notranslate nohighlight">\(1\)</span> allele IBD, then the shared allele is of type <span class="math notranslate nohighlight">\(A_1\)</span> with probability <span class="math notranslate nohighlight">\(p\)</span>, and then the other non-IBD allele, in both relatives, also needs to be <span class="math notranslate nohighlight">\(A_1\)</span> which happens with probability <span class="math notranslate nohighlight">\(p^2\)</span>, so <span class="math notranslate nohighlight">\(P(\textrm{both } A_1 A_1 | \text{1 alleles IBD})=p \times p^2\)</span>. Finally, our pair of relatives can share two alleles IBD, in which case <span class="math notranslate nohighlight">\(P(\textrm{both } A_1 A_1 | \text{2 alleles IBD}) = p^2\)</span>, because if one of our individuals is homozygous for the <span class="math notranslate nohighlight">\(A_1\)</span> allele, both individuals will be. Putting this all together our Equation <a class="reference internal" href="#equation-eq-2-7">(7)</a> becomes</p>
<div class="math notranslate nohighlight" id="equation-eq-2-8">
<span class="eqno">(8)<a class="headerlink" href="#equation-eq-2-8" title="Permalink to this equation">#</a></span>\[P(\text{both }A_{1}A_{1}) = p^{4}r_{0} + p^{3}r_{1} + p^{2}r_{2}\]</div>
<p>Note that for specific cases we could also calculate this by summing over all the possible genotypes their shared ancestor(s) had; however, that would be much more involved and not as general as the form we have derived here.</p>
<p>We can write out terms like Equation <a class="reference internal" href="#equation-eq-2-8">(8)</a> for all of the possible configurations of genotype sharing/non-sharing between a pair of individuals. Based on this we can write down the expected number of polymorphic sites where our individuals are observed to share 0, 1, or 2 alleles.</p>
<aside class="margin sidebar">
<p class="sidebar-title"></p>
<figure class="align-left" id="figure-2-12">
<img alt="../../_images/8576533889_3a131ffc4c_z.jpg" src="../../_images/8576533889_3a131ffc4c_z.jpg" />
<figcaption>
<p><span class="caption-number">Fig. 12 </span><span class="caption-text">- Florida Scrub-Jays (<em>Aphelocoma coerulescens</em>). <span style="font-size: smaller;">The birds of America : from drawings made in the United States and their territories. 1880. Audubon J.J. Image from the <a class="reference external" href="https://www.biodiversitylibrary.org/page/40447048%5C#page/169/mode/1up">Biodiversity Heritage Library</a>. Contributed by Smithsonian Libraries. Licensed under CC BY-2.0.</span></span><a class="headerlink" href="#figure-2-12" title="Permalink to this image">#</a></p>
</figcaption>
</figure>
</aside>
<div class="admonition-question-8-trickier-question admonition" id="question-2-8">
<p class="admonition-title">Question 8 (Trickier question)</p>
<p>The genotype of our suspect in <a class="reference internal" href="chapter-02.2-allele-frequencies.html#question-2-5"><span class="std std-ref">Question 5</span></a> turns out to be 100/80 for D16S539 and 70/80 at TH01. The suspect is not a match to the DNA from the crime scene; however, they could be a sibling.</p>
<p>Calculate the joint probability of observing the genotype from the crime
and our suspect:</p>
<ul class="simple">
<li><p><strong>A)</strong> Assuming that they share no close relationship.</p></li>
<li><p><strong>B)</strong> Assuming that they are full sibs.</p></li>
<li><p><strong>C)</strong> Briefly explain your findings.</p></li>
</ul>
</div>
<p>There’s a variety of ways to estimate the relationships among individuals using genetic data. An example of using allele sharing to identify relatives is offered by the work of Nancy Chen [in collaboration with Stepfanie Aguillon, see <span id="id2">[<a class="reference internal" href="../chapter-00-bibliography.html#id226" title="Stepfanie M. Aguillon, John W. Fitzpatrick, Reed Bowman, Stephan J. Schoech, Andrew G. Clark, Graham Coop, and Nancy Chen. Deconstructing isolation-by-distance: the genomic consequences of limited dispersal. PLOS Genetics, 13(8):1-27, 08 2017. URL: https://doi.org/10.1371/journal.pgen.1006911, doi:10.1371/journal.pgen.1006911.">Aguillon <em>et al.</em>, 2017</a>, <a class="reference internal" href="../chapter-00-bibliography.html#id225" title="Nancy Chen, Elissa J. Cosgrove, Reed Bowman, John W. Fitzpatrick, and Andrew G. Clark. Genomic consequences of population decline in the endangered florida scrub-jay. Current Biology, 26(21):2974 - 2979, 2016. URL: http://www.sciencedirect.com/science/article/pii/S0960982216310077, doi:https://doi.org/10.1016/j.cub.2016.08.062.">Chen <em>et al.</em>, 2016</a>]</span>. <span id="id3">Chen <em>et al.</em> [<a class="reference internal" href="../chapter-00-bibliography.html#id225" title="Nancy Chen, Elissa J. Cosgrove, Reed Bowman, John W. Fitzpatrick, and Andrew G. Clark. Genomic consequences of population decline in the endangered florida scrub-jay. Current Biology, 26(21):2974 - 2979, 2016. URL: http://www.sciencedirect.com/science/article/pii/S0960982216310077, doi:https://doi.org/10.1016/j.cub.2016.08.062.">2016</a>]</span> has collected genotyping data from thousands of Florida Scrub Jays at over ten thousand loci. These Jays live at the Archbold field site, and have been carefully monitored for many decades allowing the pedigree of many of the birds to be known. Using these data, she estimates allele frequencies at each locus. Then by equating the observed number of times that a pair of individuals share <span class="math notranslate nohighlight">\(0\)</span>, <span class="math notranslate nohighlight">\(1\)</span>, or <span class="math notranslate nohighlight">\(2\)</span> alleles to the theoretical expectation, she estimates the probability of <span class="math notranslate nohighlight">\(r_0\)</span>, <span class="math notranslate nohighlight">\(r_1\)</span>, and <span class="math notranslate nohighlight">\(r_2\)</span> for each pair of birds. A plot of these are shown in <a class="reference internal" href="#figure-2-13"><span class="std std-numref">Fig. 13</span></a>, showing how well the estimates match those known from the pedigree.</p>
<figure class="align-left" id="figure-2-13">
<img alt="../../_images/FSJ_IBD.jpg" src="../../_images/FSJ_IBD.jpg" />
<figcaption>
<p><span class="caption-number">Fig. 13 </span><span class="caption-text">- Estimated coefficient of kinship from Florida Scrub Jays. Each point is a pair of individuals, plotted by their estimated IBD (<span class="math notranslate nohighlight">\(r_1\)</span> and <span class="math notranslate nohighlight">\(r_2\)</span>) from their genetic data. The points are coloured by their known pedigree relationships. Note that most pairs have low kinship, and no recent genealogical relationship, and so appear as black points in the lower left corner. Thanks to Nancy Chen for supplying the data. Code <a class="reference external" href="https://github.com/cooplab/popgen-notes/blob/master/Rcode/FSJ_IBD/FSJ_IBD_plotting.R">here</a>.</span><a class="headerlink" href="#figure-2-13" title="Permalink to this image">#</a></p>
</figcaption>
</figure>
</section>
<section id="sharing-of-genomic-blocks-among-relatives">
<h2>Sharing of genomic blocks among relatives.<a class="headerlink" href="#sharing-of-genomic-blocks-among-relatives" title="Permalink to this headline">#</a></h2>
<figure class="align-left" id="figure-2-14">
<img alt="../../_images/First_cousin_overlap.pdf" src="../../_images/First_cousin_overlap.pdf" />
<figcaption>
<p><span class="caption-number">Fig. 14 </span><span class="caption-text">- A simulation of sharing between first cousins. The regions of your grandmother’s 22 autosomes that you inherited are coloured red, those that your cousins inherited are coloured blue. In the third panel we show the overlapping genomic regions in purple, these regions will be IBD in you and your cousin. If you are full first cousins, you will also have shared genomic regions from your shared grandfather, not shown here. Details about how we made these simulations <a class="reference external" href="https://gcbias.org/2013/12/02/how-many-genomic-blocks-do-you-share-with-a-cousin/">here</a>.</span><a class="headerlink" href="#figure-2-14" title="Permalink to this image">#</a></p>
</figcaption>
</figure>
<p>We can more directly see the sharing of the genome among close relatives using high-density SNP genotyping arrays. In <a class="reference internal" href="#figure-2-14"><span class="std std-numref">Fig. 14</span></a> we show a simulation of first cousins’ genomic sharing from their shared grandmother. Colored purple are regions where they have matching genomic material, due to having inherited it IBD from their shared grandmother.</p>
<p>First cousins will share at least one allele of your genotype at all of the polymorphic loci in these purple regions. There’s a range of methods to detect such sharing. One way is to look for unusually long stretches of the genome where two individuals are never homozygous for different alleles. By identifying pairs of individuals who share an unusually large number of such putative IBD blocks, we can hope to identify unknown relatives in genotyping datasets. In fact, companies like 23&amp;me and <a class="reference external" href="http://Ancestry.com">Ancestry.com</a> use signals of IBD to help identify family ties.</p>
<p>As another example, consider the case of third cousins. You share one of eight sets of great-great-grandparents with each of your (likely many) third cousins. On average, you and each of your third cousins each inherit one-sixteenth of your genome from each of those two great-great-grandparents. This turns out to imply that on average, a little less than one percent of your and your third cousin’s genomes (<span class="math notranslate nohighlight">\(2 \times (\frac{1}{16})^2 =0.78\%\)</span>) will be identical by virtue of descent from those shared ancestors. A simulated example where third cousins share blocks of their genome (on chromosome 16 and 2) due to their great-great-grandmother is shown in <a class="reference internal" href="#figure-2-15"><span class="std std-numref">Fig. 15</span></a>.</p>
<figure class="align-left" id="figure-2-15">
<img alt="../../_images/Third_cousin_overlap_1.png" src="../../_images/Third_cousin_overlap_1.png" />
<figcaption>
<p><span class="caption-number">Fig. 15 </span><span class="caption-text">- A simulation of sharing between third cousins, the details are the same as in <a class="reference internal" href="#figure-2-14"><span class="std std-numref">Fig. 14</span></a>.</span><a class="headerlink" href="#figure-2-15" title="Permalink to this image">#</a></p>
</figcaption>
</figure>
<aside class="margin sidebar">
<p class="sidebar-title"></p>
<p>Indeed the suspect in case of the Golden State Killer was identified through identifying third cousins that genetically matched a DNA sample from an old crime scene (see <a class="reference external" href="https://gcbias.org/2018/05/07/how-lucky-was-the-genetic-investigation-in-the-golden-state-killer-case/">here</a> for more details).</p>
</aside>
<p>Note how if you compare <a class="reference internal" href="#figure-2-15"><span class="std std-numref">Fig. 15</span></a> and <a class="reference internal" href="#figure-2-14"><span class="std std-numref">Fig. 14</span></a>, individuals inherit less IBD from a shared great-great-grandmother than from a shared grandmother, as they inherit from more total ancestors further back. Also notice how the sharing occurs in shorter genomic blocks, as it has passed through more generations of recombination during meiosis. These blocks are still detectable, and so third cousins can be detected using high-density genotyping chips, allowing more distant relatives to be identified than single marker methods alone. More distant relations than third cousins, e.g. fourth cousins, start to have a significant probability of sharing none of their genome IBD. But you have many fourth cousins, so you will share some of your genome IBD with some of them; however, it gets increasingly hard to identify the degree of relatedness from genetic data the deeper in the family tree this sharing goes.</p>
</section>
<section id="inbreeding">
<h2>Inbreeding<a class="headerlink" href="#inbreeding" title="Permalink to this headline">#</a></h2>
<p>We can define an inbred individual as an individual whose parents are more closely related to each other than two random individuals drawn from some reference population.</p>
<aside class="margin sidebar">
<p class="sidebar-title"></p>
<figure class="align-left" id="figure-2-16">
<img alt="../../_images/Child_first_cousins_Homozy_BD.png" src="../../_images/Child_first_cousins_Homozy_BD.png" />
<figcaption>
<p><span class="caption-number">Fig. 16 </span><span class="caption-text">- Alleles being transmitted through an inbred pedigree. The two sisters (mum and aunt) share two alleles identical by descent (IBD). The cousins share one pink  allele IBD. The offspring of first cousins is homozygous by descent at this locus for the pink allele. You can simulate more examples for yourself <a class="reference external" href="https://cooplab.shinyapps.io/Shiny_pedigree_sims/">here</a>.</span><a class="headerlink" href="#figure-2-16" title="Permalink to this image">#</a></p>
</figcaption>
</figure>
</aside>
<p>When two related individuals produce an offspring, that individual can receive two alleles that are identical by descent, i.e. they can be homozygous by descent (sometimes termed autozygous), due to the fact that they have two copies of an allele through different paths through the pedigree. This increased likelihood of being homozygous relative to an outbred individual is the most obvious effect of inbreeding. It is also the one that will be of most interest to us, as it underlies a lot of our ideas about inbreeding depression and population structure. For example, in <a class="reference internal" href="#figure-2-16"><span class="std std-numref">Fig. 16</span></a> our offspring of first cousins is homozygous by descent having received the same IBD allele via two different routes around an inbreeding loop.</p>
<p>As the offspring receives a random allele from each parent (<span class="math notranslate nohighlight">\(i\)</span> and <span class="math notranslate nohighlight">\(j\)</span>), the probability that those two alleles are identical by descent is equal to the kinship coefficient <span class="math notranslate nohighlight">\(F_{ij}\)</span> of the two parents (Equation <a class="reference internal" href="#equation-eq-2-5">(5)</a>). This follows from the fact that the genotype of the offspring is made by sampling an allele at random from each of our parents.</p>
<center>
<div style="width: 50%;">
<table class="colwidths-auto table" id="table-2-5">
<caption><span class="caption-number">Table 5 </span><span class="caption-text">- Generalized Hardy– Weinberg. Dropped the indices i and j for simplicity.</span><a class="headerlink" href="#table-2-5" title="Permalink to this table">#</a></caption>
<thead>
<tr class="row-odd"><th class="text-align:center head"><p><span class="math notranslate nohighlight">\(f_{11}\)</span></p></th>
<th class="text-align:center head"><p><span class="math notranslate nohighlight">\(f_{12}\)</span></p></th>
<th class="text-align:center head"><p><span class="math notranslate nohighlight">\(f_{22}\)</span></p></th>
</tr>
</thead>
<tbody>
<tr class="row-even"><td class="text-align:center"><p><span class="math notranslate nohighlight">\((1-F)p^{2}+Fp\)</span></p></td>
<td class="text-align:center"><p><span class="math notranslate nohighlight">\((1-F)2pq\)</span></p></td>
<td class="text-align:center"><p><span class="math notranslate nohighlight">\((1-F)q^{2}+Fq\)</span></p></td>
</tr>
</tbody>
</table>
</div>
</center>
<p>The only way the offspring can be heterozygous (<span class="math notranslate nohighlight">\(A_1 A_2\)</span>) is if their two alleles at a locus are not IBD (otherwise they would necessarily be homozygous). Therefore, the probability that they are heterozygous is</p>
<div class="math notranslate nohighlight" id="equation-eq-2-9">
<span class="eqno">(9)<a class="headerlink" href="#equation-eq-2-9" title="Permalink to this equation">#</a></span>\[P(A_{1}A_{2}) = P(A_{1}A_{2} \text{ | I&amp;J not IBD})P(\text{I&amp;J not IBD}) = 2pq(1-F_{ij}),\]</div>
<p>The offspring can be homozygous for the <span class="math notranslate nohighlight">\(A_1\)</span> allele in two different ways. They can have two non-IBD alleles that are not IBD but happen to be of the allelic type <span class="math notranslate nohighlight">\(A_1\)</span>, or their two alleles can be IBD, such that they inherited allele <span class="math notranslate nohighlight">\(A_1\)</span> by two different routes from the same ancestor. Thus, the probability that an offspring is homozygous for <span class="math notranslate nohighlight">\(A_1\)</span> is</p>
<div class="math notranslate nohighlight" id="equation-eq-2-10">
<span class="eqno">(10)<a class="headerlink" href="#equation-eq-2-10" title="Permalink to this equation">#</a></span>\[\begin{split}P(A_{1}A_{2}) &amp;= P(A_{1}A_{2} \text{ | I&amp;J not IBD})P(\text{I&amp;J not IBD}) + P(A_{1}A_{2} \text{ | I&amp;J IBD})P(\text{I&amp;J IBD})\\
    &amp;= p^{2}(1-F_{ij}+pF_{ij})\end{split}\]</div>
<p>using the Law of Total Probability (see Appendix Equation <code class="xref eq docutils literal notranslate"><span class="pre">eq-A.12</span></code>). Therefore, the frequencies of the three possible genotypes can be written as given in <a class="reference internal" href="#table-2-5"><span class="std std-numref">Table 5</span></a>, which provides a generalization of the Hardy–Weinberg proportions.</p>
<div class="admonition-question-9 admonition" id="question-2-9">
<p class="admonition-title">Question 9</p>
<p>The frequency of the <span class="math notranslate nohighlight">\(A_1\)</span> allele is <span class="math notranslate nohighlight">\(p\)</span> at a biallelic locus. Assume that our population is randomly mating and that the genotype frequencies in the population follow from HW. We select two individuals at random to mate from this population. We then mate the children from this cross. What is the probability that the child from this full sib-mating is homozygous?</p>
</div>
<section id="multiple-inbreeding-loops-in-a-pedigree">
<h3>Multiple inbreeding loops in a pedigree.<a class="headerlink" href="#multiple-inbreeding-loops-in-a-pedigree" title="Permalink to this headline">#</a></h3>
<p>Up to this point we have assumed that there is at most one inbreeding loop in the recent family history of our individuals, i.e. the parents of our inbred individual have at most one recent genealogical connection. However, an individual who has multiple inbreeding loops in their pedigree can be homozygous by descent thanks to receiving IBD alleles via multiple different different loops. To calculate inbreeding in pedigrees of arbitrary complexity, we can extend beyond our original relatedness coefficients <span class="math notranslate nohighlight">\(r_0\)</span>, <span class="math notranslate nohighlight">\(r_1\)</span>, and <span class="math notranslate nohighlight">\(r_2\)</span> to account for higher order sharing of alleles IBD among relatives. For example, we can ask, what is the probability that <em>both</em> of the alleles in the first individual are shared IBD with one allele in the second individual? There are nine possible relatedness coefficients in total to completely describe kinship between two diploid individuals, and we won’t go in to them here as it’s a lot to keep track of. However, we will show how we can calculate the inbreeding coefficient of an individual with multiple inbreeding loops more directly.</p>
<p>Let’s say the parents of our inbred individual (B and C) have <span class="math notranslate nohighlight">\(K\)</span> shared ancestors, i.e. individuals who appear in both B and C’s recent family trees. We denote these shared ancestors by <span class="math notranslate nohighlight">\(A_1, \dots,A_K\)</span>, and we denote by <span class="math notranslate nohighlight">\(n\)</span> the total number of individuals in the chain from B to C via ancestor <span class="math notranslate nohighlight">\(A_i\)</span>, including B, C, and <span class="math notranslate nohighlight">\(A_i\)</span>. For example, if B is C’s aunt, then B and C share two ancestors, which are B’s parents and, equivalently, C’s grandparents. In this case, there are n=4 individuals from B to C through each of these two shared ancestor. In the general case, the kinship coefficient of B and C, i.e. the inbreeding coefficient of their child, is</p>
<div class="math notranslate nohighlight" id="equation-eq-2-11">
<span class="eqno">(11)<a class="headerlink" href="#equation-eq-2-11" title="Permalink to this equation">#</a></span>\[F = \sum_{i=1}^K \frac{1}{2^{n_i}}(1+f_{A_i})\]</div>
<aside class="margin sidebar">
<p class="sidebar-title"></p>
<p>For example, in the case of our aunt-nephew case, assuming that the aunt’s two parents are their only recent shared ancestors, then <span class="math notranslate nohighlight">\(F = \frac{1}{24} + \frac{1}{24} = \frac{1}{8}\)</span>, in agreement with the answer we would obtain from Equation <a class="reference internal" href="#equation-eq-2-5">(5)</a>.</p>
</aside>
<p>where <span class="math notranslate nohighlight">\(f_{A_i}\)</span> is the inbreeding coefficient of the ancestor <span class="math notranslate nohighlight">\(A_i\)</span>. What’s happening here is that we sum over all the mutually-exclusive paths in the pedigree through which B and C can share an allele IBD. With probability <span class="math notranslate nohighlight">\(\frac{1}{2^{n_i}}\)</span>, a pair of alleles picked at random from B and C is descended from the same ancestral allele in individual <span class="math notranslate nohighlight">\(A_i\)</span>, in which case the alleles are IBD. However, even if B inherits the maternal allele and C inherits the paternal allele of shared ancestor <span class="math notranslate nohighlight">\(A_i\)</span>, if <span class="math notranslate nohighlight">\(A_i\)</span> was themselves inbred, with probability <span class="math notranslate nohighlight">\(f_{A_i}\)</span> those two alleles are themselves IBD. Thus a shared <em>inbred</em> ancestor further increases the kinship of B and C.</p>
<aside class="margin sidebar">
<p class="sidebar-title"></p>
<figure class="align-left" id="figure-2-18">
<img alt="../../_images/378px-Juan_de_Miranda_Carreno_002.jpg" src="../../_images/378px-Juan_de_Miranda_Carreno_002.jpg" />
<figcaption>
<p><span class="caption-number">Fig. 17 </span><span class="caption-text">Charles II of Spain (by Juan Carreño de Miranda, 1685). <a class="reference external" href="https://it.wikipedia.org/wiki/Carlo_II_di_Spagna%5C#/media/File:Juan_de_Miranda_Carreno_002.jpg">Public Domain</a>.</span><a class="headerlink" href="#figure-2-18" title="Permalink to this image">#</a></p>
</figcaption>
</figure>
</aside>
<figure class="align-left" id="figure-2-17">
<img alt="../../_images/Carlos_second_pedigree_2_trimmed.pdf" src="../../_images/Carlos_second_pedigree_2_trimmed.pdf" />
<figcaption>
<p><span class="caption-number">Fig. 18 </span><span class="caption-text">The pedigree of King Charles II of Spain. Pedigree from <a class="reference external" href="https://commons.wikimedia.org/wiki/File:Carlos_segundo80.png">wikimedia</a> drawn by <a class="reference external" href="https://en.wikipedia.org/wiki/User:Lec_CRP1">Lec CRP1</a>, public domain.</span><a class="headerlink" href="#figure-2-17" title="Permalink to this image">#</a></p>
</figcaption>
</figure>
<p>Multiple inbreeding loops increase the probability that a child is homozygous by descent at a locus, which can be calculated simply by plugging in <span class="math notranslate nohighlight">\(F\)</span>, the child’s inbreeding coefficient, into our generalized HW equation.</p>
<p>As one extreme example of the impact of multiple inbreeding loops in an individual’s pedigree, let’s consider king Charles II of Spain, the last of the Spanish Habsburgs. Charles was the son of Philip IV of Spain and Mariana of Austria, who were uncle and niece. If this were the only inbreeding loop, then Charles would have had an inbreeding coefficient of <span class="math notranslate nohighlight">\(\frac{1}{8}\)</span>. Unfortunately for Charles, the Spanish Habsburgs had long kept wealth and power within their family by arranging marriages between close kin. The pedigree of Charles II is shown in <a class="reference internal" href="#figure-2-17"><span class="std std-numref">Fig. 18</span></a>, and multiple inbreeding loops are apparent. For example, Phillip III, Charles II’s grandfather and great-grandfather, was himself a child of an uncle-niece marriage.</p>
<aside class="margin sidebar">
<p class="sidebar-title"></p>
<p>Pedro Gargantilla, who performed Charles’s autopsy, stated that his body “did not contain a single drop of blood; his heart was the size of a peppercorn; his lungs corroded; his intestines rotten and gangrenous; he had a single testicle, black as coal, and his head was full of water.” While some of this description may refer to actual medical conditions, some of these details seem a little unlikely. See <a class="reference external" href="https://www.thevintagenews.com/2017/03/23/when-charles-ii-of-spain-died-the-autopsy-stated-that-his-body-did-not-contain-a-single-drop-of-blood-and-his-head-was-full-of-water/">here</a>.</p>
</aside>
<p><span id="id4">Alvarez <em>et al.</em> [<a class="reference internal" href="../chapter-00-bibliography.html#id208" title="Gonzalo Alvarez, Francisco C Ceballos, and Celsa Quinteiro. The role of inbreeding in the extinction of a european royal dynasty. PLoS One, 4(4):e5174, 2009.">2009</a>]</span> calculated that Charles II had an inbreeding coefficient of <span class="math notranslate nohighlight">\(0.254\)</span>, equivalent to a full-sib mating, thanks to all of the inbreeding loops in his pedigree. Therefore, he is expected to have been homozygous by descent for a full quarter of his genome. As we’ll talk about later in these notes, this means that Charles may have been homozygous for a number of recessive disease alleles, and indeed he was a very sickly man who left no descendants due to his infertility. Thus plausibly the end of one of the great European dynasties came about through inbreeding.</p>
</section>
</section>
<section id="calculating-inbreeding-coefficients-from-genetic-data">
<h2>Calculating inbreeding coefficients from genetic data<a class="headerlink" href="#calculating-inbreeding-coefficients-from-genetic-data" title="Permalink to this headline">#</a></h2>
<p>If the observed heterozygosity in a population is <span class="math notranslate nohighlight">\(H_O\)</span>, and we assume that the generalized Hardy–Weinberg proportions hold, we can set <span class="math notranslate nohighlight">\(H_O\)</span> equal to <span class="math notranslate nohighlight">\(f_{12}\)</span>, and solve Equation <a class="reference internal" href="#equation-eq-2-9">(9)</a> for <span class="math notranslate nohighlight">\(F\)</span> to obtain an estimate of the inbreeding coefficient as</p>
<aside class="margin sidebar">
<p class="sidebar-title"></p>
<p>The hat on the F here denotes that this is an estimate from a sample.</p>
</aside>
<div class="math notranslate nohighlight" id="equation-eq-2-12">
<span class="eqno">(12)<a class="headerlink" href="#equation-eq-2-12" title="Permalink to this equation">#</a></span>\[\hat{F} = 1-\frac{f_{12}}{2pq} = \frac{2pq - f_{12}}{2pq}.\]</div>
<p>As before, <span class="math notranslate nohighlight">\(p\)</span> is the frequency of allele <span class="math notranslate nohighlight">\(A_{1}\)</span> in the population. This can be rewritten in terms of the observed heterozygosity (<span class="math notranslate nohighlight">\(H_O\)</span>) and the heterozygosity expected in the absence of inbreeding, <span class="math notranslate nohighlight">\(H_E=2pq\)</span>, as</p>
<div class="math notranslate nohighlight" id="equation-eq-2-13">
<span class="eqno">(13)<a class="headerlink" href="#equation-eq-2-13" title="Permalink to this equation">#</a></span>\[\hat{F} = \frac{H_E-H_O}{H_E} = 1 - \frac{H_O}{H_E}.\]</div>
<p>Hence, <span class="math notranslate nohighlight">\(\hat{F}\)</span> quantifies the deviation due to inbreeding of the observed heterozygosity from the one expected under random mating, relative to the latter.</p>
<div class="admonition-question-10 admonition" id="question-2-10">
<p class="admonition-title">Question 10</p>
<p>Suppose the following genotype frequencies were observed for an esterase
locus in a population of <em>Drosophila</em> (A denotes the “fast” allele and
B denotes the “slow” allele):</p>
<center>
<div style="width: 50%;">
<table class="colwidths-auto table">
<thead>
<tr class="row-odd"><th class="text-align:center head"><p><span class="math notranslate nohighlight">\(AA\)</span></p></th>
<th class="text-align:center head"><p><span class="math notranslate nohighlight">\(AB\)</span></p></th>
<th class="text-align:center head"><p><span class="math notranslate nohighlight">\(BB\)</span></p></th>
</tr>
</thead>
<tbody>
<tr class="row-even"><td class="text-align:center"><p>0.6</p></td>
<td class="text-align:center"><p>0.2</p></td>
<td class="text-align:center"><p>0.2</p></td>
</tr>
</tbody>
</table>
</div>
</center>
<p>What is the estimate of the inbreeding coefficient at the esterase locus?</p>
</div>
<aside class="margin sidebar">
<p class="sidebar-title"></p>
<figure class="align-left" id="figure-2-19">
<img alt="../../_images/5988399184_0c36a8e51c_z.jpg" src="../../_images/5988399184_0c36a8e51c_z.jpg" />
<figcaption>
<p><span class="caption-number">Fig. 19 </span><span class="caption-text">- Grey wolf (<em>Canis lupus</em>). <span style="font-size: smaller;">Dogs, jackals, wolves, and foxes: a monograph of the Canidae. 1890. y J.G. Keulemans. Image from the <a class="reference external" href="https://www.biodiversitylibrary.org/page/17002968%5C#page/58/mode/1up">Biodiversity Heritage Library</a>. Contributed by University of Toronto - Gerstein Science Information Centre. Not in copyright.</span></span><a class="headerlink" href="#figure-2-19" title="Permalink to this image">#</a></p>
</figcaption>
</figure>
</aside>
<p>If we have multiple loci, we can replace <span class="math notranslate nohighlight">\(H_O\)</span> and <span class="math notranslate nohighlight">\(H_E\)</span> by their means over loci, <span class="math notranslate nohighlight">\(\bar{H}_O\)</span> and <span class="math notranslate nohighlight">\(\bar{H}_E\)</span>, respectively. Note that, in principle, we could also calculate <span class="math notranslate nohighlight">\(F\)</span> for each individual locus first, and then take the average across loci. However, this procedure is more prone to introducing a bias if sample sizes vary across loci, which is not unlikely when we are dealing with real data.</p>
<p>Genetic markers are commonly used to estimate inbreeding for wild and/or captive populations of conservation concern. As an example of this, consider the case of the Mexican wolf (<em>Canis lupus baileyi</em>), a sub-species of gray wolf.</p>
<p>They were extirpated in the wild during the mid-1900s due to hunting, and the remaining five Mexican wolves in the wild were captured to start a breeding program. &#64;vonHoldt:11 estimated the current-day, average expected heterozygosity to be <span class="math notranslate nohighlight">\(0.18\)</span>, based on allele frequencies at over forty thousand SNPs. However, the average Mexican wolf individual was only observed to be heterozygous at <span class="math notranslate nohighlight">\(12\%\)</span> of these SNPs. Therefore, the average inbreeding coefficient for the Mexican wolf is <span class="math notranslate nohighlight">\(\hat{F} = 1 -\frac{0.12}{0.18}\)</span>, i.e. <span class="math notranslate nohighlight">\(\sim 33 \%\)</span> of a lobo’s genome is homozygous due to recent inbreeding in their pedigree.</p>
<section id="genomic-blocks-of-homozygosity-due-to-inbreeding">
<h3>Genomic blocks of homozygosity due to inbreeding.<a class="headerlink" href="#genomic-blocks-of-homozygosity-due-to-inbreeding" title="Permalink to this headline">#</a></h3>
<p>As we saw above, close relatives are expected to share alleles IBD in large genomic blocks. Thus, when related individuals mate and transmit alleles to an inbred offspring, they transmit these alleles in big blocks through meiosis. As an example, let’s return to the case of our hypothetical first cousins from <code class="xref std std-numref docutils literal notranslate"><span class="pre">figure-2.10</span></code>. If this pair of individuals had a child, one possible pattern of genetic transmission is shown in <a class="reference internal" href="#figure-2-20"><span class="std std-numref">Fig. 20</span></a>. The child has inherited the red stretch of chromosome via two different routes through their predigree from the grandparents. This is an example of an autozygous segment, where the child is homozygous by descent at all of the loci in this red region. The inbreeding coefficient of the child sets the proportion of their genome that will be in these autozygous segments. For example, a child of first full cousins is expected to have <span class="math notranslate nohighlight">\(1/16\)</span> of their genome in these segments. The more distant the loop in the pedigree, the more meioses that chromosomes have been through and the shorter individual blocks will be. A child of first cousins will have longer blocks than a child of second cousins, for example.</p>
<figure class="align-left" id="figure-2-20">
<img alt="../../_images/first_cousin_offspring.pdf" src="../../_images/first_cousin_offspring.pdf" />
<figcaption>
<p><span class="caption-number">Fig. 20 </span><span class="caption-text">- A pedigree showing the offspring of first cousins. The chromosomes of their great-grandparents are coloured different colours so their transmission can be tracked. The child is homozygous by descent (HBD) for a section of the red chromosome.</span><a class="headerlink" href="#figure-2-20" title="Permalink to this image">#</a></p>
</figcaption>
</figure>
<p>Individuals with multiple inbreeding loops in their family tree can have a high inbreeding coefficient due to the combined effect of many small blocks of autozygosity. For example, Charles II had an inbreeding coefficient that is equivalent to that of the child of full-sibs, with a quarter of his genome expected to homozygous by descent, but this would be made up of many shorter blocks.</p>
<p>We can hope to detect these blocks by looking for unusually long genomic runs of homozygosity (ROH) sites in an individual’s genome. One way to estimate an individual’s inbreeding coefficient is then to total up the proportion of an individual’s genome that falls in such ROH regions. This estimate is called <span class="math notranslate nohighlight">\(F_{ROH}\)</span>.</p>
<p>An example of using <span class="math notranslate nohighlight">\(F_{ROH}\)</span> to study inbreeding comes from the work of <span id="id5">Sams and Boyko [<a class="reference internal" href="../chapter-00-bibliography.html#id217" title="Aaron J Sams and Adam R Boyko. Fine-scale resolution of runs of homozygosity reveal patterns of inbreeding and substantial overlap with recessive disease genotypes in domestic dogs. G3: Genes, Genomes, Genetics, pages g3–200836, 2018.">2018</a>]</span>, who identified runs of homozygosity in 2,500 dogs, ranging from 500kb up to many megabases. <a class="reference internal" href="#figure-2-21"><span class="std std-numref">Fig. 21</span></a> shows the distribution of <span class="math notranslate nohighlight">\(F_{ROH}\)</span> of individuals in each dog breed for the X and autosome. In <a class="reference internal" href="#figure-2-23"><span class="std std-numref">Fig. 23</span></a> this is broken down by the length of ROH segments.</p>
<figure class="align-left" id="figure-2-21">
<img alt="../../_images/dogs_FROH.pdf" src="../../_images/dogs_FROH.pdf" />
<figcaption>
<p><span class="caption-number">Fig. 21 </span><span class="caption-text">- The distribution of FROH of individuals from various dog breeds from <span id="id6">Sams and Boyko [<a class="reference internal" href="../chapter-00-bibliography.html#id218" title="Aaron J Sams and Adam R Boyko. Fine-scale resolution and analysis of runs of homozygosity in domestic dogs. bioRxiv, 2018. URL: https://www.biorxiv.org/content/early/2018/05/07/315770, arXiv:https://www.biorxiv.org/content/early/2018/05/07/315770.full.pdf, doi:10.1101/315770.">2018</a>]</span>, licensed under CC BY 4.0.</span><a class="headerlink" href="#figure-2-21" title="Permalink to this image">#</a></p>
</figcaption>
</figure>
<aside class="margin sidebar">
<p class="sidebar-title"></p>
<figure class="align-left" id="figure-2-22">
<img alt="../../_images/14752595581_4330377c97_z.jpg" src="../../_images/14752595581_4330377c97_z.jpg" />
<figcaption>
<p><span class="caption-number">Fig. 22 </span><span class="caption-text">- English bulldog. The dogs of Boytown. 1918. Dyer, W. A.</span><a class="headerlink" href="#figure-2-22" title="Permalink to this image">#</a></p>
</figcaption>
</figure>
</aside>
<figure class="align-left" id="figure-2-23">
<img alt="../../_images/dog_FROH_dist.pdf" src="../../_images/dog_FROH_dist.pdf" />
<figcaption>
<p><span class="caption-number">Fig. 23 </span><span class="caption-text">- Cumulative density of ROH length, measured in megabases (Mb) from <span id="id7">Sams and Boyko [<a class="reference internal" href="../chapter-00-bibliography.html#id218" title="Aaron J Sams and Adam R Boyko. Fine-scale resolution and analysis of runs of homozygosity in domestic dogs. bioRxiv, 2018. URL: https://www.biorxiv.org/content/early/2018/05/07/315770, arXiv:https://www.biorxiv.org/content/early/2018/05/07/315770.full.pdf, doi:10.1101/315770.">2018</a>]</span> for various dog breeds (li- censed under CC BY 4.0). Note that longer lengths of ROH are on the left of the plot.</span><a class="headerlink" href="#figure-2-23" title="Permalink to this image">#</a></p>
</figcaption>
</figure>
<p>Dog breeds have been subject to intense breeding that has resulted in high levels of inbreeding. Of the population samples examined, Doberman Pinschers have the highest levels of their genome in runs of homozygosity (<span class="math notranslate nohighlight">\(F_{ROH}\)</span>), somewhat higher than English bulldogs. In <a class="reference internal" href="#figure-2-23"><span class="std std-numref">Fig. 23</span></a>, we can see that English bulldogs have more short ROH than Doberman Pinschers, but that Doberman Pinschers have more of their genome in very large ROH (<span class="math notranslate nohighlight">\(&gt;16 Mb\)</span>). This suggests that English bulldogs have had long history of inbreeding as they have many small blocks, but that Doberman Pinschers have a lot of recent inbreeding as their autozygosity is contained in long blocks relatively unbroken by recombination.</p>
</section>
</section>
</section>

    <script type="text/x-thebe-config">
    {
        requestKernel: true,
        binderOptions: {
            repo: "binder-examples/jupyter-stacks-datascience",
            ref: "master",
        },
        codeMirrorConfig: {
            theme: "abcdef",
            mode: "python"
        },
        kernelOptions: {
            kernelName: "python3",
            path: "./chapters/chapter-02"
        },
        predefinedOutput: true
    }
    </script>
    <script>kernelName = 'python3'</script>

              </div>
              
            </main>
            <footer class="footer-article noprint">
                
    <!-- Previous / next buttons -->
<div class='prev-next-area'>
    <a class='left-prev' id="prev-link" href="chapter-02.2-allele-frequencies.html" title="previous page">
        <i class="fas fa-angle-left"></i>
        <div class="prev-next-info">
            <p class="prev-next-subtitle">previous</p>
            <p class="prev-next-title">Allele Frequencies</p>
        </div>
    </a>
    <a class='right-next' id="next-link" href="chapter-02.4-chapter-summary.html" title="next page">
    <div class="prev-next-info">
        <p class="prev-next-subtitle">next</p>
        <p class="prev-next-title">Chapter Summary</p>
    </div>
    <i class="fas fa-angle-right"></i>
    </a>
</div>
            </footer>
        </div>
    </div>
    <div class="footer-content row">
        <footer class="col footer"><p>
  
    By Graham Coop<br/>
  
      &copy; Copyright 2022.<br/>
</p>
        </footer>
    </div>
    
</div>


      </div>
    </div>
  
  <!-- Scripts loaded after <body> so the DOM is not blocked -->
  <script src="../../_static/scripts/pydata-sphinx-theme.js?digest=1999514e3f237ded88cf"></script>


  </body>
</html>